IBO2019_DATA = {}

IBO2019_DATA["FIGURE_IDS"] = ["22"]

IBO2019_DATA[
    "Morning_1"
] = r"""
<question id="q0" points="0.0"><title id="eafa242e78e845b18d6aa5b58aab7454">Morning 1st</title><pagebreak id="facf998cc5f74225bc049ffc05661e8d" /><part id="4ac372d092bc454c888aaa289544e346">Cell Biology</part><section id="561d2288dea247e4bb1cbb29750c10ec">Q1</section><paragraph id="f25a6f2ecbd94762ab3111ab6af60b49">&lt;p&gt;The nematode &lt;em&gt;C. elegans&lt;/em&gt; develops as either a male having only one sex chromosome (XO) or a hermaphrodite with two sex chromosomes (XX). When maintained in groups containing both sexes, hermaphrodites live significantly longer than males. Two signalling pathways, the IGF-1 signalling and the sex determination pathways play pivotal roles in the control of &lt;em&gt;C. elegans&lt;/em&gt; aging.&lt;/p&gt;

&lt;p&gt;DAF-2 and DAF-16 are elements of the IGF-1 signalling pathway. Loss-of-function mutations in their genes were introduced, and the lifespan of generated transgenic animals of the two sexes were measured (Figure 1)&lt;/p&gt;</paragraph><paragraph id="bc1ca6c7e0834cf8b2ccd7a6e9cf4f7d">In all graphs, NS = not significant difference; *** = significant difference.</paragraph><figure figid="22" id="aa7c4337d981444ea4ecb53de3fa0c51" width="1"><caption id="fffac9c0da8346879c42cca46fc192b1">Fig.1. Axis = Mean lifespan (days). WT = Wild-type.</caption></figure><paragraph id="22d6f675d7434c2bba2eabfa6a24ad84">&lt;p&gt;To test whether the nematode sex-determination cascade is involved in the sex-specific regulation of aging, researchers modified the activity of two elements of this pathway, TRA-1 and TRA-3. The mutations of transgenic hermaphrodites were as follows in Figure 2.&lt;/p&gt;

&lt;ul&gt;
\t&lt;li&gt;MUT1 - TRA-1-/-&lt;/li&gt;
\t&lt;li&gt;MUT2 - TRA-1 low activity, with some function&lt;/li&gt;
\t&lt;li&gt;MUT3 - TRA-1 hyperactive function&lt;/li&gt;
\t&lt;li&gt;MUT4 - TRA-3 loss of function&lt;/li&gt;
\t&lt;li&gt;MUT5 - TRA-1 hyperactive function AND TRA-3 loss of function&lt;/li&gt;
&lt;/ul&gt;</paragraph><figure figid="22" id="2f99e2fdf21b486bb546a5313c15533c" width="1"><caption id="f66420420e66499493a0959b4f529f35">Fig.2. Axis = Mean lifespan (days). WT = Wild-type.</caption></figure><paragraph id="f1df7c0ed764473c8345acd61b5c3017">TRA-1 was identified as a transcription factor, and to assess the functionality of a potential TRA-1-binding site in the DNA at the DAF-16 gene. The expression of DAF-16 was therefore measured in the various TRA-1 mutants (Figure 3).</paragraph><figure figid="22" id="9b223cec609149dc8c0938851a9f7079" width="1"><caption id="0c189cb5b2ba45f39ba8a43aaee614b5">Fig.3. Axis = Relative DAF-16 expression level.</caption></figure><subquestion id="9a6186e4d0274a9fac06dd02d82aa1bf" part_nr="Q" points="0.0" question_nr="1.1"><paragraph id="5a2cb033c1ab46aab1ac8411206dd422">&lt;p&gt;Based on the information above, create a model of protein interactions that affect lifespan (LS). Use the options below (A-D) to fill in the boxes in the flowchart and indicate in the circles by the arrows whether the interaction is stimulatory (put &lt;strong&gt;+&lt;/strong&gt;) or inhibitory (put &lt;strong&gt;-&lt;/strong&gt;).&lt;/p&gt;

&lt;p&gt;Options:&lt;/p&gt;

&lt;p&gt;A. TRA-1&lt;/p&gt;

&lt;p&gt;B. TRA-3&lt;/p&gt;

&lt;p&gt;C. DAF-2&lt;/p&gt;

&lt;p&gt;D. DAF-16&lt;/p&gt;</paragraph><figure figid="22" id="1f00cbced09645a6b086e2294573d762" width="0.75" /></subquestion><pagebreak id="2c33b9e03958451fbc80d08fa06ef9e8" /><section id="abb231bf4b8b4aa1ab15ce8687ee25d0">Q2</section><paragraph id="806381d2bc994850bee36b118466328e">The ATP content of an average muscle fibre at rest is very low. Under short, high energy demand exercise (e.g. a 100 m sprint), the initially very low ATP levels in the muscle fibres are buffered by phosphocreatine (PCr, Figure 1).</paragraph><figure figid="22" id="3186cb8ab53e472ea2e78a7b78900767" width="0.75"><caption id="4638ba7fbd2049bf81cb0eca97289535">Fig.1</caption></figure><paragraph id="81b235854197444ea1f821650c58a619">We can measure levels of phosphocreatine and ATP in a working tissue using 31P NMR spectroscopy, through the height of the corresponding NMR peak. Figure 2 shows the NMR-spectroscopy results of muscle intracellular fluid from a subject who had done 2 minutes of vigorous exercise. A: before exercise, B: first minute of exercise, C: last minute of exercise, D: after exercise; I, II and III are peaks that represent the three phosphate groups in ATP molecules, IV: PCr, V: Pi, VI: phospho-monoesters.</paragraph><figure figid="22" id="acfeb0b7576f4f4885a9f927b517d28c" width="0.5"><caption id="a90d9c47f44042f2bb99ac1c61a1defd">Fig.2</caption></figure><paragraph id="351e7cdccbb34bcc9dea4241f44d4866">&lt;p&gt;Immediately when the actin-myosin interaction starts producing ADP, the activity of the enzyme adenylate kinase, which catalyses the near-equilibrium reaction 2 ADP \u2194 ATP + AMP becomes significant. Besides the generation of the negligible amount of available ATP, the reaction is essential because both ADP and AMP can allosterically upregulate glycolytic enzymes \u2013 and therefore prepare the muscle for longer exercise.&lt;/p&gt;</paragraph><paragraph id="6a38fa05db054de0b866e5926d133e7e">Based on the information above, indicate with an X if each of the following statements is true (T) or false (F).</paragraph><subquestion id="2249cf7bc4c8404383585edbba1397f6" part_nr="Q" points="0.0" question_nr="2.1"><paragraph id="ae563814e99a495691744f33725d462b">Hydrolysis of the high energy bond of PCr can be used directly as energy source of enzymes.</paragraph></subquestion><subquestion id="6a1c90851b634a409b48a626fed4578a" part_nr="Q" points="0.0" question_nr="2.2"><paragraph id="2fb21a8210464800a898d686c968f502">The relatively constant ATP concentrations shown on the NMR results are due to the buffering effect of PCr.</paragraph></subquestion><subquestion id="cd5518c99e814e378cfd11705dc140ad" part_nr="Q" points="0.0" question_nr="2.3"><paragraph id="330765572c544c91a58b1416e7ddfa20">The phosphocreatine system is the main energy source in marathon runners.</paragraph></subquestion><subquestion id="3ddf43ff54fb476b926e616aed2a10a0" part_nr="Q" points="0.0" question_nr="2.4"><paragraph id="f2f16060f0224457b0ae8894241e912f">AMP and ADP levels are similarly sensitive indicators of energy status (as shown by ATP concentration).</paragraph></subquestion><pagebreak id="244a0699d82842dcbc07e033b908bb9a" /><section id="2093637dfb9645f58a968446180fa4b7">Q3</section><paragraph id="9a424d436fdc48ba81b22bfb6e644744">Abscisic acid (ABA)-and salicylic acid (SA) are involved in stomatal closure. It\u2019s known that the Ca2+-independent protein kinase OST1 and Ca2+-dependent protein kinases (CPK3 and CPK6) are key for abscisic acid induced activation of an anion channel SLAC1 and stomatal closure. SLAC1 has two phosphorylation sites: S59 and S120 are both serines. Coexpression of the CPKs or OST1 together with SLAC1 genes in an observed cell leads to the following result:</paragraph><figure figid="22" id="5b2e27f136dc47f496a1048231e47da9" width="1"><caption id="16034765ca624554a38fdf987db5c6c7">Table 1. Col 1 = Expression of . Col 2 = Phosphorylation of . Col 3 = Current. An = Anionic. \u2018+\u2019 = expressed; \u2018\u2013\u2018 = not expressed; \u2018@\u2019 = not happens; \u2018***\u2019 = happens</caption></figure><paragraph id="7d54dd0b6dfa458f83371e87da108996">&lt;p&gt;The effect of salicylic acid (SA) on stomata closure of wild type (WT) &lt;em&gt;Arabidopsis thaliana&lt;/em&gt; was determined in a standardized way. Salicylic acid was added to a leaf fragments and stomata opening was measured. The stomatal assay was also carried out with some KO-mutant lines. The results are shown below.&lt;/p&gt;
(*\u2013 significant change, ns \u2013 non-significant change)</paragraph><figure figid="22" id="d6418dce8ee448ada4ad51c924f775a1" width="0.5"><caption id="6a1fb9f61b26478e80646bbabf62993f">Fig.1. Axis = Stomatal aperture (\u03bcm). WT = Wild-type.</caption></figure><figure figid="22" id="e0b3b0be45c840bba249e2bfb77a8389" width="0.5"><caption id="9a74d0764f1141c499b29cbd1f5be647">Fig.2. Axis = Stomatal aperture (\u03bcm). WT = Wild-type.</caption></figure><paragraph id="9ca69dda9fb94248b55957c82949e7de">&lt;p&gt;For the following questions indicate your answer with an &lt;strong&gt;X.&lt;/strong&gt;&lt;/p&gt;

&lt;p&gt;A. CPK3&lt;/p&gt;

&lt;p&gt;B. CPK6&lt;/p&gt;

&lt;p&gt;C. OST1&lt;/p&gt;

&lt;p&gt;D. either CPK3 or CPK6&lt;/p&gt;

&lt;p&gt;E. either CPK3 or OST1&lt;/p&gt;

&lt;p&gt;F. either CPK6 or OST1&lt;/p&gt;

&lt;p&gt;G. CPK3 and CPK6&lt;/p&gt;

&lt;p&gt;H. CPK3 and OST1&lt;/p&gt;
I. CPK6 and OST1</paragraph><subquestion id="38df06977c2d43bcb9bfc25f96dd9688" part_nr="Q" points="0.0" question_nr="3.1"><paragraph id="16bb45ed09284a0da231fb97cf80263a">Which of the kinase(s) (A-I) are required for the phosphorylation of the two sites in SALC1?</paragraph><table columns="|l|c|c|" id="7a2f686f8f7742cebebee3487114c427" top_line="1"><row bottom_line="1" id="4361583c8b794979aeea0318b410f5c6" multiplier="1"><cell id="4a72b9104eee403f83ee5535441c5f32">Q.3.1.1. S59</cell><cell id="cb734ec8ac164c1db223082a07a8b196" /></row><row bottom_line="1" id="d37a248e351c4a40a4589d0d05f928dd" multiplier="1"><cell id="792e0b6e5bd24e0b997e22ade7fba782">Q.3.1.2. S120</cell><cell id="19130ca422414069b49d0c9190c04f73" /></row></table></subquestion><subquestion id="abcf62e3431c4c93b68468962ccc2c53" part_nr="Q" points="0.0" question_nr="3.2"><paragraph id="7df24dca18fe4fdfa5d5a4437a60e532">Which of the kinase(s) (A-I) is required for salicylic acid dependent stomata closure?</paragraph></subquestion><subquestion id="4d4e18769a7543e0b441f9b886f9075b" part_nr="Q" points="0.0" question_nr="3.3"><paragraph id="43b16e35d2c84f1d9c5dda60181a4f38">&lt;p&gt;Which site or sites need to be phosphorylated to open the SALC1 anion channel?&lt;/p&gt;

&lt;p&gt;A. S59&lt;/p&gt;

&lt;p&gt;B. S120&lt;/p&gt;

&lt;p&gt;C. S59 or S120&lt;/p&gt;

&lt;p&gt;D. S59 and S120&lt;/p&gt;</paragraph></subquestion><pagebreak id="6d78f5e5f0854fb380ef159e869d8309" /><section id="83204ba5cc9e44e79109f5bbfc801f60">Q4</section><paragraph id="9fbe7ec63fec443b9df188eb700d4fdb">&lt;p&gt;Each year, the WHO forecasts which H and N antigens the dominant flu strain is likely to carry to make vaccines. In 2017, an outbreak of strains H1N1 and H3N2 was forecast for the following year. However, in 2018, H1N2 actually became dominant&lt;/p&gt;</paragraph><paragraph id="65f7ad47b46c47dabe6bf65ba42288eb">Three people (1-3) are vaccinated, and their antibodies are used in immunodiffusion assays. Virus and antibodies are loaded into neighbouring wells of an agar plate according to the table below. They diffuse towards each other and precipitate in a visible band if they crosslink.</paragraph><table columns="|l|c|c|c|" id="defe902cfaac4d7f920c8d2e1aaf434e" top_line="1"><row bottom_line="1" id="6e5daec3039e416faf019a7953e6af7b" multiplier="1"><cell id="1ea3e422c1814087aeee9c7ba592ba49">Antibody</cell><cell id="3a05ab2a75b34a67ae8b550529c19223">Well A</cell><cell id="48e8bbf370fd408c922253c90d8717cc">Well B</cell></row><row bottom_line="1" id="b6483c131cf544e2a8db3dd7a9a81e4b" multiplier="1"><cell id="ec87f5bbb4524dd5a998314b7e7086b0">1</cell><cell id="6520d08644e24a7daf83f5b2527e208b">H1N1</cell><cell id="dba45738cef24a4f966d05ee9bb4b7a4">H1N2</cell></row><row bottom_line="1" id="93928f711f8d418ebc800af66ba79f4f" multiplier="1"><cell id="1ae393bd80e94386af210e5690d425a3">2</cell><cell id="5d2a93353e2f47aa9ca962cabbbddeaf">H1N1</cell><cell id="68f9b48765804f0eb2ff9df4383fa1e9">H3N2</cell></row><row bottom_line="1" id="bb1f033e0ec247c08b93a8b2d8805e7e" multiplier="1"><cell id="20c0085bfe8f4dcd97a72f1d7e1e3753">3</cell><cell id="ca12ae334db4460cb59e98d52aa9fc4d">H1N1</cell><cell id="47f3074b2e9e499da1a120c38bb72712">H1N2</cell></row></table><paragraph id="d0442da5f35f4dc88acd22e8b19fe817">The results of the experiment are presented in Figure 1.</paragraph><figure figid="22" id="6959c24596304221a031b1946bbeae66" width="0.5"><caption id="b867112bcba3425d8fe5e0000656bc54">Fig.1</caption></figure><paragraph id="82713c1a2e404075a3fc26350ac40f2b">Indicate with an X if each the following statements are true (T) or false (F).</paragraph><subquestion id="3bbe4826efe5425a88a8b60d7e9e6b95" part_nr="Q" points="0.0" question_nr="4.1"><paragraph id="7bbe452308374ab0bd789414606044ee">The developed vaccine provided protection for antigens predicted by the WHO.</paragraph></subquestion><subquestion id="98fe63ae8aa94e3e863c2cdb752b6616" part_nr="Q" points="0.0" question_nr="4.2"><paragraph id="5811b207ec5c4429bc4f7a4422fc3072">Antibody 1 and 2 can from the same subject.</paragraph></subquestion><subquestion id="012b2c8be0ac49899e639dbcc62be256" part_nr="Q" points="0.0" question_nr="4.3"><paragraph id="bc2686e494fb481aae1558c644ed79a8">Patient 2 is protected against the new H1N2 influenza outbreak.</paragraph></subquestion><subquestion id="acc821efedaf4d11be017e320d6f7dcb" part_nr="Q" points="0.0" question_nr="4.4"><paragraph id="beb3d43a64c442fe9c64f63f7fddbdaa">Based on the available information mark antigens the patients are definitely immunised against with an X. Mark all other boxes with O.</paragraph><table columns="|l|c|c|c|c|c|" id="2877bcb4c7a04c3e96d37c091e886b4c" top_line="1"><row bottom_line="1" id="103de828d6954f96a5f89ed941a0093a" multiplier="1"><cell id="64697fe7fdbe4ca0be260cfa6c81b761">Antibody</cell><cell id="374db1bec6c44b5387c21fba7025a7db">H1</cell><cell id="8552ee24940546a2b6ee31ef74ba9580">H3</cell><cell id="4e2f8748b1de42da8e0d704956f7830e">N1</cell><cell id="81f37f4f460746178a69d1941e3902dc">N2</cell></row><row bottom_line="1" id="3ce564a1a4264816bd88e8bf61d52225" multiplier="1"><cell id="0ae0fc4f0ff24197b4e08c1ffd425db0">1</cell><cell id="17cdb01f60b840baa487fe5ba09dec88" /><cell id="4df7612867a54aeb8fad8151498ed013" /><cell id="74aecbafb83d45c8ab3b5cce56f6eb2e" /><cell id="8b12240fb4b0479480c299a4f92376bc" /></row><row bottom_line="1" id="f93da3c681e74daaac6a33f48ccd81b0" multiplier="1"><cell id="030e98978b9e4cd89f7aa5669ad84f2d">2</cell><cell id="c458ef2c900a4b76ace38eaf603d449c" /><cell id="eafcfc36be7f4ec8b4dd3ac8d77c62ff" /><cell id="259651bbedbc419a92730da53f624db0" /><cell id="500f1e8952764e069379b0835d8af07e" /></row><row bottom_line="1" id="047cda1800274a7e89f32a3d543e795e" multiplier="1"><cell id="cb87cfa4e7ae4a6da3fc22f8817663ca">3</cell><cell id="e7290584af094bd29013e0d356eef4f3" /><cell id="f0cbe62abdc646bbac0b3861c9c7a20a" /><cell id="e0ea10e8dca44495abae44de6aec4920" /><cell id="1e8735aadbb349fead677a0aab311ae0" /></row></table></subquestion><paragraph id="59885d2b67cb403b9d1fd86d760b89c5">&lt;p&gt;Catumaxomab is a new anti-cancer antibody, unlike natural antibodies, binds two different antigens.&lt;/p&gt;

&lt;p&gt;Indicate with an X if each the following statements are true (T) or false (F).&lt;/p&gt;</paragraph><subquestion id="81f5021e83f34ce1b04b7d6d008e1087" part_nr="Q" points="0.0" question_nr="4.5"><paragraph id="f7ceee888863480b97ef4e01eb5b3582">Catumaxomab in an immunodiffusion assay with purified CD3 protein in both antigen wells will give precipitation similar to Antibody 1 in Figure 1.</paragraph></subquestion><subquestion id="5174c9a4fbb6426888f86b68619f4400" part_nr="Q" points="0.0" question_nr="4.6"><paragraph id="a36bac49b276401fb50307a1493a77f8">The different specificity chains of Catumaxomab are held together by covalent bonds.</paragraph></subquestion><pagebreak id="5f6d92147c344c2099dfe4b4c2a26ce3" /><section id="512f6654f5554da783f15bd359daaeb1">Q5</section><paragraph id="4f3f2c66f8df4385885577ba00b9e11f">In 1963, a series of experiments were performed by John Cairns and peers to test the mechanism of DNA replication. An experimental organism with circular dsDNA chromosomes was grown for several generations in a medium containing radiolabelled [&lt;span class="math-tex"&gt;\\(^3H\\)&lt;/span&gt;] thymidine. The organism used these bases to replicate their circular dsDNA chromosomes and therefore the subsequently synthesised DNA became radiolabelled. Samples were rapidly frozen during DNA synthesis and then observed so radiolabelled bases appeared as a line of dark beads on a string in SEM photographs. They found that each circular chromosome contains a single \u201ceye\u201d during replication, resembling the Greek letter \u03b8 (theta structures, Figure 1).</paragraph><figure figid="22" id="444bde01cb164914a18bcdcc9b654cec" width="0.75"><caption id="eed1dfccc1a141f5988e42b92827d7f8">Fig.1. Autoradiogram and its interpretive drawing of a replicating chromosome. Legend 1 = Replication eye</caption></figure><paragraph id="daa16bdf781442a29f68a9c335ba9f76">In a second experiment, researchers grown the cells in a medium that contained both non-labelled and [&lt;span class="math-tex"&gt;\\(^3H\\)&lt;/span&gt;] thymidine but the latter was only present at low concentrations. Then, during the replicative period, they increased the concentration of [&lt;span class="math-tex"&gt;\\(^3H\\)&lt;/span&gt;] thymidine in the medium and immediately isolated samples to freeze and photograph. The results are shown in Figure 2.</paragraph><figure figid="22" id="f8aa08670b3b4d3699322e666cdc87b5" width="0.75"><caption id="3ffb9dcc198148ddaabf3347eeb2edd2">Fig.2. Radiograph of the DNA</caption></figure><paragraph id="3492d6de868646fc9a5f2defbdf92a5a">Indicate with an X if each the following statements are true (T) or false (F).</paragraph><subquestion id="2b5c1d2697374895bfe87490e9113c6d" part_nr="Q" points="0.0" question_nr="5.1"><paragraph id="53531558096c4c8d97c8d7a88a6d3769">The experiments show that there is a single replication origin in this organism.</paragraph></subquestion><subquestion id="3af2929de30a4613bdcf2af4536fce66" part_nr="Q" points="0.0" question_nr="5.2"><paragraph id="888cd45f09fb402c9c64f9774f6af0bd">The experiments suggest bidirectional replication.</paragraph></subquestion><subquestion id="4362e28a6db24c9e9c997e3c99450287" part_nr="Q" points="0.0" question_nr="5.3"><paragraph id="7ce71c964a5c4c2d9bd5695bd7f4f4c5">&lt;p&gt;Which of the following theories of DNA replication is consistent with the results above? Indicate with an X.&lt;/p&gt;

&lt;p&gt;A. Conservative&lt;/p&gt;

&lt;p&gt;B. Semi-conservative&lt;/p&gt;

&lt;p&gt;C. Dispersive&lt;/p&gt;

&lt;p&gt;D. None of the above&lt;/p&gt;</paragraph></subquestion><subquestion id="96662645ffc640edac38f52e964b0487" part_nr="Q" points="0.0" question_nr="5.4"><paragraph id="f378f46c14e847bdb504c91c0e839616">&lt;p&gt;What can you conclude from the results on the relative abundance of thymidine? Indicate with an X.&lt;/p&gt;

&lt;p&gt;A. Thymidine is more abundant in the replication origin(s), than in other area of the DNA.&lt;/p&gt;

&lt;p&gt;B. Thymidine is less abundant in the replication origin(s), than in other area of the DNA.&lt;/p&gt;

&lt;p&gt;C. Thymidine is equally abundant in the replication origin(s) and in other area of the DNA.&lt;/p&gt;

&lt;p&gt;D. Relative thymidine abundancy of the replication origin(s) cannot be inferred from these results without doubt.&lt;/p&gt;</paragraph></subquestion><pagebreak id="8f68be626e1547a58f183cfd0ccab8a4" /><section id="cf9fcc1ad026437cb4514417c853b67a">Q6</section><paragraph id="030a4e6a96fc42e7bc093f19ca80888a">&lt;p&gt;During DNA replication, RNA primers are synthesised, which are then extended by DNA polymerase. On the lagging strand, DNA polymerase is then released, forming an Okazaki fragment, and the process repeats.&lt;/p&gt;
The lagging strand was immobilised at one end to a flow cell. The other end is attached to a florescent bead. Liquid is moved across the cell to stretch the DNA and the position of the bead is measured and plotted as 'DNA length change'.</paragraph><figure figid="22" id="92dcfcddf20d42ae831e6e266dd2078f" width="1"><caption id="3b02cfb347a24f0ab11c2a76585592fa">Fig.1. Axis 1 = Time (s). Axis 2 = DNA length change (kb). Legend = Primer.</caption></figure><subquestion id="4ba618675f7d4a2bb3a035949769f61f" part_nr="Q" points="0.0" question_nr="6.1"><paragraph id="9afc6e11a2a446ce847a70f6bdb7c4b9">Indicate with an X which position on the graph (I-III) may correspond to which step of DNA replication (A-C).</paragraph><figure figid="22" id="d75388e347434ecd939e1f11127ada8a" width="0.75" /></subquestion><subquestion id="497f8275d43f4ac0b14cf24226f2d40b" part_nr="Q" points="0.0" question_nr="6.2"><paragraph id="61c48b3a4e304df99dd23698a4d2cdea">How many base pairs long is the Okazaki fragment based on the figure (to the nearest 500bp)?</paragraph></subquestion><subquestion id="f2545bfc8e3c4609b0acdeee14bf0d89" part_nr="Q" points="0.0" question_nr="6.3"><paragraph id="bb6aa47964fc423787b626e339d40075">What is the rate of DNA synthesis? (in bp/sec)</paragraph></subquestion><subquestion id="90f2dcf334dd4d598b29e08f2ac49ee9" part_nr="Q" points="0.0" question_nr="6.4"><paragraph id="6ca0258ae3de4f62bd6c997a84923861">1U of Taq polymerase is defined as the amount of enzyme required to incorporate 10nmol dNTP into DNA in 30 minutes under optimal conditions (72\u00b0C). In a PCR mixture (25\u00b5L) made with 1U of Taq pol, assuming unlimited template DNA and optimal conditions throughout, how long would it theoretically take for dNTPlevels to drop from a starting 400\u00b5M concentration to 200\u00b5M in minutes?</paragraph></subquestion><pagebreak id="6793df77b4584eaab31ce87bdcf79173" /><section id="3dfaed0248ca4ffebecc9d20f5d8dcaf">Q7</section><paragraph id="ffd84dd53b2346c6af3c52df883bca21">You find a mutation in p53 ('the guardian of the genome') at nucleotide 42. In your sequence, it is an adenine (A), whereas it is usually guanine (G) in the wild-type, as shown in Figure 1.</paragraph><figure figid="22" id="dc7fae792eb8419cb41c31b4dfabd27c" width="1"><caption id="77aaf6ab18fc44f99568bed71a40dd4c">Fig.1</caption></figure><subquestion id="97d34b09f177495aa3fd63786493d58a" part_nr="Q" points="0.0" question_nr="7.1"><paragraph id="f4efae9c8be94c359cd4bc49c6939914">&lt;p&gt;You design a test for the mutant allele using restriction enzymes. Indicate with an X which enzyme (A-D) can be used to distinguish patient samples?&lt;/p&gt;

&lt;p&gt;A. MboII&lt;/p&gt;

&lt;p&gt;B. BglII&lt;/p&gt;

&lt;p&gt;C. DpnI&lt;/p&gt;

&lt;p&gt;D. HgaI&lt;/p&gt;</paragraph></subquestion><paragraph id="8070d13abead405d85bccfac97358f76">A clinician uses the appropriate enzyme to carry out restriction fragment length polymorphism analysis. Results for patient A-G are shown in Figure 2.</paragraph><figure figid="22" id="87c04053bdeb414da013ba49abe8bb97" width="0.75"><caption id="8fb09bc6990c4ceeb87e1a5df7735e3a">Fig.2</caption></figure><subquestion id="7d787206ad0a4d8d92b5c573747f777d" part_nr="Q" points="0.0" question_nr="7.2"><paragraph id="2a90a6389cc04ae18588722305c5e20f">Determine the genotype of the patients by marking in it with an X. The letters in the table refer to the nucleotides found at position 42.</paragraph><table columns="|l|c|c|c|c|c|c|c|c|" id="e3c91172afea478bb48070f18ceb45a9" top_line="1"><row bottom_line="1" id="6a434f6655a442f1bc456f75c58140ff" multiplier="1"><cell id="efd85269f87648a287e036ffbdc1ff3f">Nucleotide 42</cell><cell id="2c270c30e7ad4023ba70d35e0e72a30c">A</cell><cell id="ef7adca900134316ba49384c716ce3ba">B</cell><cell id="d16e90db897944bbb1e3f92498d5cf62">C</cell><cell id="c1bcd516386642949903a9acbce9c1bb">D</cell><cell id="02c2d2cf529b435789bc2b72541618ed">E</cell><cell id="0805c82c36304a28ab77e1ab7901c40e">F</cell><cell id="e963cf1f9e7d4dd3afedf37ac6c1c2c3">G</cell></row><row bottom_line="1" id="b853cfecfefc498db6bde1e886dc36fa" multiplier="1"><cell id="ed5b06fd811949209be7cd9b22a1be3b">GG</cell><cell id="101eac12936e4b5fa298b511c5a05432" /><cell id="35d43cabc2194faeaf562b9c9cb4fbea" /><cell id="60aa688137314fe8924407ff3281cf26" /><cell id="9441ebdbe12241d6bfd05cef00cc91af" /><cell id="557e0649a7da4472ae1157f6336165fa" /><cell id="0d28c3dd8d1d44bdaa7be55336837bd5" /><cell id="df2fa70f3a734426b26ab22ea37f1db6" /></row><row bottom_line="1" id="aa77763198524a82aaab32a35799916d" multiplier="1"><cell id="6b28891e776745c98075c66226d6715a">AG</cell><cell id="dd450c562fc842709ded3b1b24072b3d" /><cell id="03f28d765ba14321a8b617466622fc13" /><cell id="9cd479200efa4a109e99075428643b9e" /><cell id="6803091dbc514ce9a0098022546c1a27" /><cell id="3f52308032784f3892b7dbd513110d4d" /><cell id="6b389aa1252e4c6d9b16726de86e8fb7" /><cell id="0b462f979820490bab038267f238726c" /></row><row bottom_line="1" id="86c08853804e4fed9aff64a06762a9d9" multiplier="1"><cell id="45ec773df0b1461394b09f6d74101fc8">AA</cell><cell id="4cbbc01723a34db7aff4c23b4fc290f5" /><cell id="6f8dae6ac2034280ad75e5643654f2b2" /><cell id="ded4735b9a054f4dbc39dfdce80ec8c6" /><cell id="29e62545c944483fb14e0544af3126ff" /><cell id="1621459bdc3c425c9da45debfab77276" /><cell id="fadcb276b4464a74bed8270c6c84756f" /><cell id="d5255e34ae914e2989f60628353f1855" /></row></table></subquestion><paragraph id="006c77af5cba46b1974ccdd25790bf30">Following genotyping you look at p53 transcript and protein levels as seen in Figure 3.</paragraph><figure figid="22" id="f4d081d543ce43c3bcaa2e2cf40cc931" width="0.75"><caption id="a1490c0d6e6545f2adff1675fe54c586">Fig.3. Legend 1 = RNA. Legend 2 = Protein.</caption></figure><subquestion id="1f29181ab96e4d9d93fa5a1add7b527a" part_nr="Q" points="0.0" question_nr="7.3"><paragraph id="eaadd941f5534e38bd489c3d42188f30">&lt;p&gt;Indicate with an X whether the mutation affects:&lt;/p&gt;

&lt;p&gt;A. p53 transcription&lt;/p&gt;

&lt;p&gt;B. p53 translation&lt;/p&gt;

&lt;p&gt;C. Both&lt;/p&gt;</paragraph></subquestion><subquestion id="278e6eaeb6524e0285857c669d4783a9" part_nr="Q" points="0.0" question_nr="7.4"><paragraph id="c46f59d77c9a43a98dc40dca5dc401a7">Normally, when the genome is damaged, four molecules of p53 form a complex which can then bind to DNA as a transcription factor. In an experiment a cell is heterozygous for a mutation which produces p53 that prevents DNA binding. If the amount of transcription is proportional to the level of active p53, what fraction of transcription occurs in this cell compared to normal cells?</paragraph></subquestion><pagebreak id="8077460b3523452f9004cf1765702291" /><part id="744f964b6aad48fd8bf6300ed3934f4b">Animal Anatomy and Physiology</part><section id="21bcf7fe12d342628027dcc659d98036">Q8</section><paragraph id="e5d1d76c8bde4e1f8166af8262961498">In an experiment, the breathing tube of the spirometer was filled with oxygen and fitted with a carbon dioxide absorber that removed all exhaled &lt;span class="math-tex"&gt;\\(CO_2\\)&lt;/span&gt;, as illustrated in Figure 1. Test subject \u201cA\u201d was allowed to breath from the apparatus for 4 minutes. The result of this experiment is presented in Figure 2.</paragraph><figure figid="22" id="1f73c6f88da64887a6be85c3e8586514" width="0.5"><caption id="5e5efd09aeb8440389714e7a6bf09064">Fig.1. Legend 1 = Kymograph. Legend 2 = Water level. Legend 3 = Mouthpiece. Legend 4 = Nose clip. Legend 5 = Carbon dioxide absorber. Legend 6 = Counterpoise. Legend 7 = Spirometer chamber.</caption></figure><figure figid="22" id="b6ff1b916dc749db8942793ce739adb2" width="0.6"><caption id="5d6bc1428417459f9084f2972f0b2148">Fig.2. Spirometer experiment. Axis 1 = Time (min). Axis 2 = Spirometer volume (L).</caption></figure><subquestion id="d42d7b6d53d64439a13bab2688e0a05e" part_nr="Q" points="0.0" question_nr="8.1"><paragraph id="27563cb0ace340b9b28d7f28bda78b22">What is the tidal volume of test subject \u201cA\u201d? (in mL)</paragraph></subquestion><subquestion id="fdebcd5e17524133a4592f42a236498f" part_nr="Q" points="0.0" question_nr="8.2"><paragraph id="8c1bb962fe7d4f349bd2d8f3f6492e9d">What is the breathing frequency of test subject \u201cA\u201d? (in breaths/min)</paragraph></subquestion><subquestion id="031cbf9fd12648d497b51a3e1008c2d5" part_nr="Q" points="0.0" question_nr="8.3"><paragraph id="d30c3c21c94e438f99654d1327227102">What is the oxygen consumption of test subject \u201cA\u201d? (in mL/min)</paragraph></subquestion><subquestion id="7f1e3bdda5784a39965e567a72adb23e" part_nr="Q" points="0.0" question_nr="8.4"><paragraph id="165ed04cf4d342d599477448be1ed736">&lt;p&gt;In an experiment where the &lt;span class="math-tex"&gt;\\(CO_2\\)&lt;/span&gt; absorber is removed, how would the test subject\u2019s breathing frequency change? Indicate your answer by putting an X in the appropriate box on the answer sheet.&lt;/p&gt;

&lt;p&gt;A. Increase&lt;/p&gt;

&lt;p&gt;B. Not change&lt;/p&gt;

&lt;p&gt;C. Decrease&lt;/p&gt;</paragraph></subquestion><paragraph id="28c5324f57724832b277f39c4fe4aa38">In a different spirometry experiment, we measured the volume and speed of air during normal and forced exhalation and inhalation.</paragraph><figure figid="22" id="dc7009ad5cc54d14bdfe0c3847113e58" width="0.5"><caption id="355e309cfbed4d6daac59cddd59b83ed">Fig.3. Axis = Volume (ml/kg).</caption></figure><paragraph id="105707323a424b7a891dc49faf192444">&lt;ul&gt;
\t&lt;li&gt;TLC - Total lung capacity: the volume in the lungs at maximal inflation, the sum of VC and RV.&lt;/li&gt;
\t&lt;li&gt;TV - Tidal volume: that volume of air moved into or out of the lungs during quiet breathing&lt;/li&gt;
\t&lt;li&gt;RV - Residual volume: the volume of air remaining in the lungs after a maximal exhalation&lt;/li&gt;
\t&lt;li&gt;ERV - Expiratory reserve volume: the maximal volume of air that can be exhaled from the end-expiratory position&lt;/li&gt;
\t&lt;li&gt;IRV - Inspiratory reserve volume: the maximal volume that can be inhaled from the end-inspiratory level&lt;/li&gt;
\t&lt;li&gt;VC - Vital capacity: the volume of air breathed out after the deepest inhalation.&lt;/li&gt;
\t&lt;li&gt;FRC - Functional residual capacity: the volume in the lungs at the end-expiratory position&lt;/li&gt;
&lt;/ul&gt;</paragraph><paragraph id="6e065e1d855040928fd4cd3ffe97e006">&lt;p&gt;Consider test subject \u201cB\u201d who had the following results:&lt;/p&gt;

&lt;ul&gt;
\t&lt;li&gt;VC = 5,200 mL&lt;/li&gt;
\t&lt;li&gt;IRV = 3,300 mL&lt;/li&gt;
\t&lt;li&gt;FRC = 2,500 mL&lt;/li&gt;
\t&lt;li&gt;RV = 1,300 mL&lt;/li&gt;
&lt;/ul&gt;</paragraph><subquestion id="e2251b0e5dad4d218f067658fbaa2f9c" part_nr="Q" points="0.0" question_nr="8.5"><paragraph id="25f3b8a1af224332a375b32a9b2cd542">Calculate the tidal volume of test subject \u201cB\u201d and give your answer on the answer sheet in millilitres (mL).</paragraph></subquestion><paragraph id="efdf1be4fce845e2928f257c151c0c9f">Test subject \u201cC\u201d was also tested. After several measurements, researchers found that her tidal volume is 500 mL and respiratory rate is 15/min when resting. During intense exercise her &lt;span class="math-tex"&gt;\\(O_2\\)&lt;/span&gt; consumption rises leading to a 3-fold higher volume of air inhaled per minute and an increased respiratory rate of 25/min.</paragraph><subquestion id="b40c977368bf410dba513942c9ef8e8f" part_nr="Q" points="0.0" question_nr="8.6"><paragraph id="4cbec7b26c3c426993560a590019e5ae">Based on the graph above and the data provided, calculate the tidal volume of test subject \u201cC\u201d during exercise. Give your answer on the answer sheet in millilitres (mL).</paragraph></subquestion><pagebreak id="071ed12a3af346febec5cd31490471de" /><section id="e822b22a58264479bffeee5b33a212e9">Q9</section><paragraph id="f3c22d06f1c94ec7b37484f5f7784729">In the brainstem there is a neuronal regulation network which maintains the rhythmic breathing as it can be seen in the list below.</paragraph><paragraph id="7809f253972f49eaa3620b9404885178">&lt;p&gt;Pneumotaxic center (PC):&lt;/p&gt;

&lt;ul&gt;
\t&lt;li&gt;Controls the duration of inhalation by signalling to inspiratory area&lt;/li&gt;
\t&lt;li&gt;Strong activity &lt;span class="math-tex"&gt;\\(\\longrightarrow\\)&lt;/span&gt; length of inspiration &amp;lt; 0.5 seconds&lt;/li&gt;
\t&lt;li&gt;Weak activity &lt;span class="math-tex"&gt;\\(\\longrightarrow\\)&lt;/span&gt; length of inspiration &amp;gt; 5 seconds&lt;/li&gt;
&lt;/ul&gt;

&lt;p&gt;Apneostic center (AC):&lt;/p&gt;

&lt;ul&gt;
\t&lt;li&gt;Promotes inhalation by stimulating the DRG&lt;/li&gt;
\t&lt;li&gt;Inhibited by pulmonary stretch receptors&lt;/li&gt;
\t&lt;li&gt;Inhibits PC&lt;/li&gt;
&lt;/ul&gt;

&lt;p&gt;Dorsal respiratory group (DRG):&lt;/p&gt;

&lt;ul&gt;
\t&lt;li&gt;Controls the basic rhythm by triggering respiratory impulses&lt;/li&gt;
\t&lt;li&gt;Vagal and glossopharyngeal nerves bring peripheral chemosensory information to it&lt;/li&gt;
&lt;/ul&gt;

&lt;p&gt;Ventral respiratory group (VRG):&lt;/p&gt;

&lt;ul&gt;
\t&lt;li&gt;Inactive during normal baseline respiration&lt;/li&gt;
\t&lt;li&gt;High DRG activity (increased need for ventilation) stimulates it&lt;/li&gt;
\t&lt;li&gt;Has neurons for both inhalation and exhalation&lt;/li&gt;
\t&lt;li&gt;Inhibits AC&lt;/li&gt;
&lt;/ul&gt;

&lt;p&gt;Central chemosensitive area (CC):&lt;/p&gt;

&lt;ul&gt;
\t&lt;li&gt;Stimulated by high blood &lt;span class="math-tex"&gt;\\(pCO_2\\)&lt;/span&gt;&lt;/li&gt;
\t&lt;li&gt;Excites other respiratory centres&lt;/li&gt;
&lt;/ul&gt;</paragraph><paragraph id="7e478f72ddbb418ab9da67c9f7e4a63d">&lt;p&gt;Decide if the following treatments would result in an increase , no change or decrease in the described feature. Indicate your answer by putting an X in the appropriate box on the answer sheet.&lt;/p&gt;

&lt;ul&gt;
\t&lt;li&gt;A. Increase&lt;/li&gt;
\t&lt;li&gt;B. No change&lt;/li&gt;
\t&lt;li&gt;C. Decrease&lt;/li&gt;
&lt;/ul&gt;</paragraph><subquestion id="b8ed5061476244b3ab2475f457bdea4d" part_nr="Q" points="0.0" question_nr="9.1"><paragraph id="c16970895b7647a0bd4a813e174093ce">Respiratory rate upon damage to pneumotaxic center</paragraph></subquestion><subquestion id="0fe035c862ac4a758685cea6bced3a1d" part_nr="Q" points="0.0" question_nr="9.2"><paragraph id="bd2d3b2ab7a0404d92a39d1e06655b58">Blood pH after a stroke affecting central chemosensitive area neurons</paragraph></subquestion><subquestion id="3a9ce43a4c4341d9ab8b114d1993df1a" part_nr="Q" points="0.0" question_nr="9.3"><paragraph id="95e6c261cfa04b669bf01504411dc819">Ventral respiratory group activity upon a deep inhalation</paragraph></subquestion><subquestion id="90bcae45ae8749ca8f5147460e9af489" part_nr="Q" points="0.0" question_nr="9.4"><paragraph id="86782447ac114c7b9b80566c3016d8c9">Central chemosensitive area neuron membrane hyperpolarisation upon voluntarily holding one\u2019s breath</paragraph></subquestion><subquestion id="25e292f18c364e9c878702a87d047e25" part_nr="Q" points="0.0" question_nr="9.5"><paragraph id="7dda74ead5ab47ef8d36b50615d1d841">Length of inhalation upon overstimulation of apneostic center</paragraph></subquestion><pagebreak id="880ae8c9b74349a19f127b701e2de7fe" /><section id="97bb73cd818848a3afd3b50638357815">Q10</section><paragraph id="5e2d7b57aaa7489698c488e5e1ee3cb6">&lt;p&gt;Pressure\u2013volume work (or PV work) occurs when the volume of a system changes. It can be calculated as the product of \u0394 volume and \u0394 pressure. The work of breathing is mainly determined by the compliance of the lungs, which is defined as:&lt;/p&gt;

&lt;p&gt;Compliance = \u0394 volume (mL) /\u0394 pressure (cm &lt;span class="math-tex"&gt;\\(H_2O\\)&lt;/span&gt;)&lt;/p&gt;</paragraph><paragraph id="df47d707e8994ccdbc7f077e63e36af0">Compliance itself is dependent on two main factors, (1) the elastic characteristics of the lung, and (2) the surface tension generated by the air-liquid interface at the inner alveolar surface. In 1929, von Neergaard excised the lung of a cat which he then inflated and then deflated using air and then inflated and deflated using saline solution. The results are plotted in the figure below.</paragraph><figure figid="22" id="084367e685644593874e6d8ecf730e2f" width="0.5"><caption id="b2e410abc4bd4a29bfb869a35c4d3423">Fig.1. Axis 1 = Pressure (cm &lt;span class="math-tex"&gt;\\(H_2O\\)&lt;/span&gt;). Axis 2 = Volume (ml). Legend 1 = Saline inflation. Legend 2 = Air inflation. Empty circles &lt;span class="math-tex"&gt;\\(\\circ\\)&lt;/span&gt; : inflation. Full dots &lt;span class="math-tex"&gt;\\(\\bullet\\)&lt;/span&gt; : deflation.</caption></figure><paragraph id="7be814f39af0487d9f7ec6e4799c7c88">For the following pairs of statements (A and B), evaluate the mathematical relationship between the values. Indicate your answers using the following symbols &gt; or &lt; or = in the cells provided on the answer sheet.</paragraph><subquestion id="9a3f192df5a840eb999975ad9d674cfe" part_nr="Q" points="0.0" question_nr="10.1"><paragraph id="9340d78777234357b95ce39c0872294b">&lt;p&gt;A: Work of inflating the cat lung with saline solution.&lt;/p&gt;

&lt;p&gt;B: Work of inflating the cat lung with air.&lt;/p&gt;</paragraph></subquestion><subquestion id="b2ecdbb7bfc64fe2ab49de2cdd754218" part_nr="Q" points="0.0" question_nr="10.2"><paragraph id="421f9967d0d1450eb6965e0a918a8fcb">&lt;p&gt;A: The compliance of the lung at volumes above 35 mL (upon normal air inhalation).&lt;/p&gt;

&lt;p&gt;B: The compliance of the lung at volumes below 35 mL (upon normal air inhalation).&lt;/p&gt;</paragraph></subquestion><subquestion id="4e0e80cf84054dc0beabfdbe0db796dc" part_nr="Q" points="0.0" question_nr="10.3"><paragraph id="5c2be9b3c44d434cb6dfb0051e95035a">&lt;p&gt;A: Contribution of intrinsic elastic forces to total lung compliance.&lt;/p&gt;

&lt;p&gt;B: Contribution of surface tension to total lung compliance.&lt;/p&gt;</paragraph></subquestion><subquestion id="58b2be72e72e42af8920bff32c4f14b9" part_nr="Q" points="0.0" question_nr="10.4"><paragraph id="3ec933a949a440ab92211f5a49b5b409">&lt;p&gt;A: Compliance of a healthy, normally functioning lung.&lt;/p&gt;

&lt;p&gt;B: Compliance of a lung in which surfactant production is abolished by genetic mutations.&lt;/p&gt;</paragraph></subquestion><subquestion id="ca3129a9a59c4a0d9773e64bd85c5692" part_nr="Q" points="0.0" question_nr="10.5"><paragraph id="78224dd1f2704da8aced52ddbdc63059">&lt;p&gt;A: Total lung capacity in a normally functioning cat lung.&lt;/p&gt;

&lt;p&gt;B: Total lung capacity in the same cat\u2019s lung after chemical inhibition of surfactant production.&lt;/p&gt;</paragraph></subquestion><pagebreak id="1b5cc389c90f4e4d9784eb10b825c972" /><section id="6ac54e363f124c2c9a79fb554f5ae507">Q11</section><paragraph id="f639c74c29d64c86839b6476531bbd0c">The electromyogram (EMG) is a technique used to measure the collective electrical activity of active skeletal muscle fibres using electrodes placed on the skin. A human subject was prepared for a muscle electrophysiology experiment to test the conduction velocity of their motor neurons. The experimental setup and a trace of one of their measurements are illustrated below.</paragraph><figure figid="22" id="633391a0a07b43fc8ab9051c01658fcc" width="0.5"><caption id="3a4cd775986e4b078396e433071cc56b">Fig.1. Legend 1 = Connected to amplifier and detector. Legend 2 = Stick-on electrodes. Legend 3 = Stimulator applied to notch in back of elbow. Legend 4 = To isolated stimulator</caption></figure><figure figid="22" id="a3b8eaff24e54b52bb6fb2ba11f2e9ac" width="1"><caption id="dacf031032d3498ebda7bcd70ef54676">&lt;p&gt;Fig.2. Axis 1 = Time (sec). Axis 2 = Force transducer (N). Axis 3 = Electromyogram (mV).&lt;/p&gt;
&lt;em&gt;(The spike in the \u201cElectromyogram\u201d trace at \u201cA\u201d (time = 0.03 sec) is added by the computer to show when the stimulus was presented.)&lt;/em&gt;</caption></figure><subquestion id="4f718bb79e984e68b25f19a4044ae9dc" part_nr="Q" points="0.0" question_nr="11.1"><paragraph id="97436be2342045f7977156bbd1f8f3ae">Calculate the conduction velocity of the motor neuron. Assume that the initiation and transmission of signalling within the muscle cells take zero milliseconds. The distance between the stimulation electrode and the EMG electrode was 420 mm.  Give your answer in m.sec-1 and round it to the nearest whole number.</paragraph></subquestion><paragraph id="c508a44ea792402aa604c5da3fc34e96">&lt;p&gt;For the following questions, answer with the letters of the processes listed below (A-I). Note that not all of the options are actually involved in the illustrated process. Indicate your answer by putting an X in the appropriate box on the answer sheet.&lt;/p&gt;

&lt;p&gt;A. &lt;span class="math-tex"&gt;\\(Ca^{2+}\\)&lt;/span&gt; channel opening in the membrane of the sarcoplasmic reticulum&lt;/p&gt;

&lt;p&gt;B. Acetylcholine released&lt;/p&gt;

&lt;p&gt;C. &lt;span class="math-tex"&gt;\\(Ca^{2+}\\)&lt;/span&gt; binding to troponin&lt;/p&gt;

&lt;p&gt;D. Action potential initiated in muscle fibre membrane&lt;/p&gt;

&lt;p&gt;E. Action potential propagated along motor neuron&lt;/p&gt;

&lt;p&gt;F. Actin-myosin cross-bridge cycling&lt;/p&gt;

&lt;p&gt;G. Troponin blockage of myosin binding sites restored&lt;/p&gt;

&lt;p&gt;H. Active transport of &lt;span class="math-tex"&gt;\\(Ca^{2+}\\)&lt;/span&gt; across membrane of sarcoplasmic reticulum&lt;/p&gt;

&lt;p&gt;I. &lt;span class="math-tex"&gt;\\(Ca^{2+}\\)&lt;/span&gt; concentration increases in the muscle fibre\u2019s cytoplasm&lt;/p&gt;</paragraph><subquestion id="227f3c377200407e9389dc35319262f5" part_nr="Q" points="0.0" question_nr="11.2"><paragraph id="31c0912e5a934ffaac7f9c5bf93821c6">Which &lt;u&gt;&lt;strong&gt;2&lt;/strong&gt;&lt;/u&gt; processes happen between time points A and B?</paragraph></subquestion><subquestion id="18851ed488884f04b243b055cafbb626" part_nr="Q" points="0.0" question_nr="11.3"><paragraph id="c8c7b6cc6bf04c029d158fd810cb109f">What happens immediately at point B?</paragraph></subquestion><subquestion id="7aca7d55aa71494bba814192183e9b56" part_nr="Q" points="0.0" question_nr="11.4"><paragraph id="9145549cddc74ef0b2cbd185bf4a698c">Which process accounts for the observation of the peak labelled C?</paragraph></subquestion><subquestion id="4fd3fb8c294b4fd9a455eb059043688d" part_nr="Q" points="0.0" question_nr="11.5"><paragraph id="62fc44c2a61a4a1cbb36dfae9ea8a696">Which process is most likely to account for the slowness of the fall of the trace after point C?</paragraph></subquestion><pagebreak id="6c618893328649cea4160604848d6ed2" /><section id="41c6ef0a818446889bbec0127743f1ca">Q12</section><paragraph id="a0259187da074132819d26d9f7192d43">Nitric oxide (NO) plays a role in physiological regulation of the circulatory system. NO is the activator of soluble guanylyl cyclase, leading to the formation of cyclic GMP (cGMP), an important second messenger in vascular smooth muscle.</paragraph><figure figid="22" id="b70757668c1a4db08bd4749fd9084749" width="0.75"><caption id="32376dfba0334f439215eb219ca97ed5">Fig.1. Legend 1 = Adrenaline. Legend 2 = Endothelial Cell. Legend 3 = Smooth Muscle Cell. Legend 4 = Sarcoplasmic Reticulum.</caption></figure><paragraph id="fbde93c9206649ba92dc8c71ace0f349">&lt;p&gt;The contraction of vascular smooth muscle determines the diameter of a blood vessel. The relationship between vessel diameter, pressure, and flow is described by Poisseuille\u2019s law.&lt;/p&gt;</paragraph><equation id="fcc5ab7ed21345338a8c6082e5b5403f">Q= \\frac{ \\pi  r^{4}  \\Delta P}{8  \\mu L}</equation><paragraph id="a2debb2c6aa94f178590e2a78bbfb0ee">&lt;p&gt;Where:&lt;/p&gt;

&lt;ul&gt;
\t&lt;li&gt;Q is flow in mL/min&lt;/li&gt;
\t&lt;li&gt;r is the vessel radius&lt;/li&gt;
\t&lt;li&gt;\u2206P is the pressure gradient&lt;/li&gt;
\t&lt;li&gt;\u03bc is the viscosity of the fluid in the vessel&lt;/li&gt;
\t&lt;li&gt;L is the length of the vessel&lt;/li&gt;
&lt;/ul&gt;</paragraph><paragraph id="bd840b9b33364977a096b0bf23bd7c6c">&lt;p&gt;Based on the above, how would the specified characteristics be changed in the following scenarios? For the &lt;u&gt;&lt;strong&gt;next three questions&lt;/strong&gt;&lt;/u&gt;, decide if the result is increase, no change or decrease. Indicate your answer by putting an X in the appropriate box on the answer sheet.&lt;/p&gt;

&lt;p&gt;A. Increase&lt;/p&gt;

&lt;p&gt;B. No change&lt;/p&gt;

&lt;p&gt;C. Decrease&lt;/p&gt;</paragraph><subquestion id="e35cd183e4ed43bfa0086572e1290dbf" part_nr="Q" points="0.0" question_nr="12.1"><paragraph id="30591ca6adea4f3b81ef161c4b4c4e6e">Flow in the vessel in case of fight-or-flight response in a living organism.</paragraph></subquestion><subquestion id="432e6728fe074b9ca2a50bc99fa05adb" part_nr="Q" points="0.0" question_nr="12.2"><paragraph id="fcc76ee382de414f92554a488bd8befa">Flow in the vessel upon administration of the drug sildenafil, which inhibits cGMP specific phosphodiesterase (PDE).</paragraph></subquestion><subquestion id="6177c0158fa74515bc5cd2f7c07898cc" part_nr="Q" points="0.0" question_nr="12.3"><paragraph id="422f02d27bdd429d9cc3fb73ad1ad094">Membrane potential of the smooth muscle cell upon adrenaline infusion into a perfused vessel preparation.</paragraph></subquestion><subquestion id="e615afb4ca78475881227a515d47c440" part_nr="Q" points="0.0" question_nr="12.4"><paragraph id="d758a715001d4760937ae4760e459b5e">What fold increase of pressure results in an equal increase in flow as a 10-fold increase in radius? Provide your answer in the appropriate box on the answer sheet.</paragraph></subquestion><pagebreak id="68cc72d2ef5f4a12bbf7a007eb8b4cff" /><section id="1773b765f1434f61a98da61fd3708830">Q13</section><paragraph id="8a5178c3644648b590b5320fcdaed502">The water content of the human body is about 60%. The distribution of this huge amount of fluid and their content are both important to understand the main processes of homeostasis. Figure 1 shows the amount of body fluid compartment and their ionic content.</paragraph><figure figid="22" id="d0b5e231d5be471a86d5855b4e02ded1" width="0.75"><caption id="82392cc3fd7f4ea6b70516c029c560ae">&lt;p&gt;Fig.1. Distribution of the different fluid compartments in a healthy adult. Axis 1 = Mass (kg). Axis 2 = Osmotic concentration (mOsmol/kg). Legend 1 = Intracellular fluid. Legend 2 = Interstitial fluid. Legend 3 = Blood plasma.&lt;/p&gt;</caption></figure><paragraph id="5c5a118c540c4525be1099d64e010dbf">Inadequate supply of water and ions or the abnormal loss of these substances alter the values found in Figure 1. In Figure 2, four physiological conditions with their distinctive values are shown, in which changes occur in the mass and/or osmotic concentration of body fluids. The diagrams only show the intracellular (dark grey) and the extracellular (light grey) fluid compartments and their mass and osmotic concentration values. The areas surrounded by the dashed lines represent the normal values of the compartments shown in the previous diagram.</paragraph><figure figid="22" id="45aa808cf6a94babbe83464d9abc1d6f" width="0.75"><caption id="be366c31044e49cab32272d824bda66e">Fig.2. Axis 1 = Mass )kg). Axis 2 = Osmotic concentration (mOsmol/kg)</caption></figure><subquestion id="03c5f52853424c2687fcd73a5f4021a1" part_nr="Q" points="0.0" question_nr="13.1"><paragraph id="8dec2c525f934a3abfaf8868c00f80d6">Calculate the extracellular fluid/intracellular fluid mass ratio of sodium ions based on normal values. Assume, that the concentration of sodium ion in the extracellular fluid is 140 mOsm/kg and in the intracellular fluid 10 mOsm/kg. Give your answer to the nearest integer.</paragraph></subquestion><paragraph id="73c8ada89c754470af719d745db2c42a">Match the following statements describing different physiological conditions with the letters in Figure 2 (A-D). Indicate your answer by putting an X in the appropriate box (A-D) on the answer sheet.</paragraph><subquestion id="50a80284a2ca45a3b4fca56f4cd77de0" part_nr="Q" points="0.0" question_nr="13.2"><paragraph id="3efd748076b64584a3d725ad8bc4957f">A condition typical after drinking large quantities of diluted liquid.</paragraph></subquestion><subquestion id="62e7aa6d9bc84b3e8ad5c2c809ed7318" part_nr="Q" points="0.0" question_nr="13.3"><paragraph id="6f64ccc69b544256ad79d29f52b01a47">A condition typical of dehydration (loss of water).</paragraph></subquestion><subquestion id="7303311e190344d9a40e9cb378d0adbf" part_nr="Q" points="0.0" question_nr="13.4"><paragraph id="280d8ae29d594080bb4dd4ae88b21db0">A condition brought about after severe vomiting and diarrhoea.</paragraph></subquestion><subquestion id="b7fa3cfa8d3f4037af37bb119c6fba28" part_nr="Q" points="0.0" question_nr="13.5"><paragraph id="c6aa442dae844d1f9ab251428a360fb8">A condition most likely to cause the blood pressure to increase.</paragraph></subquestion><subquestion id="7b6f10b72b4345fcbbe81c9ba8bd1e22" part_nr="Q" points="0.0" question_nr="13.6"><paragraph id="515b7a75779245afbfc9690eee5db716">Which of these conditions will inhibit release of vasopressin (ADH) into the blood?</paragraph></subquestion><pagebreak id="eca76c29e0974cad8f6402da25cd9a9b" /><section id="539dccdc9d1a46f893add66fb875d196">Q14</section><paragraph id="43eeb4d9e9794cf6aedccba901e46101">Insulin is a 51-amino acid protein consisting of 2 polypeptide chains linked by disulphide bonds.</paragraph><figure figid="22" id="2dd2a6f96fe5449e80c26da0705a45bc" width="1"><caption id="37473c32b7e04d97b6aa0aa30cf2a52f">Fig.1. Legend 1 = A chain. Legend 2 = B chain</caption></figure><paragraph id="f9d6f34fd05341c983cfcf5183d33a84">In type 1 diabetes mellitus, there is no adequate insulin release due to the loss of beta-cells of the pancreas. Therefore, the only treatment is insulin injection, which is most commonly administered into the fat under the skin. Here it forms hexamers, which then dissociate to be absorbed into the blood stream.</paragraph><paragraph id="3cf5d63cf92b4d05921b86b9205379a0">&lt;p&gt;The graph shows the plasma level of three available insulins (1-3) after subcutaneous injection:&lt;/p&gt;

&lt;p&gt;1. Human insulin produced in &lt;em&gt;Saccharomyces cerevisiae&lt;/em&gt; by recombinant DNA technology.&lt;/p&gt;

&lt;p&gt;2. An insulin analogue in which swapping one amino acid reduces dimer and hexamer formation.&lt;/p&gt;

&lt;p&gt;3. An insulin analogue in which amino acid changes to the A and B chains change the molecule\u2019s isoelectric point from 5.7 to a more neutral pH.&lt;/p&gt;</paragraph><figure figid="22" id="53ad869841374997971e0acc03110ee9" width="0.75"><caption id="4c6884eb0e584553a43057ae925f4622">Fig.2. Axis 1 = Time. Axis 2 = Relative plasma insulin level</caption></figure><subquestion id="85803e0688d348eab2edd18748f4d74b" part_nr="Q" points="0.0" question_nr="14.1"><paragraph id="ce9dfbab4a764d62a0b8644994eeb7db">Match the insulin types (1-3) with the plots (A-C) based on the information above. Indicate your answer by putting an X in the appropriate box on the answer sheet.</paragraph></subquestion><paragraph id="9b3bcd90367e46f29f2fedcc86dceae3">Indicate with an X if each of the following statements is true (T) or false (F).</paragraph><subquestion id="b7bfd383b0404653bf5360029bd4f3ea" part_nr="Q" points="0.0" question_nr="14.2"><paragraph id="afbeae9c300a46e9bda79db90e6f6510">Oral administration (e.g. in the form of pills) of insulin is approximately as effective as subcutaneous administration.</paragraph></subquestion><subquestion id="83b92efe9c7b472885ace7a1d7ca623d" part_nr="Q" points="0.0" question_nr="14.3"><paragraph id="59c2a74b09804dcc85ef62b39b2912e5">Stimulation of insulin release from pancreatic beta cells can be an alternative therapy in type 1 diabetes mellitus.</paragraph></subquestion><subquestion id="69acee37d4294d10965f3240ed604a86" part_nr="Q" points="0.0" question_nr="14.4"><paragraph id="3490aa137f8f4a8a965cfb0dd3dbe684">Immunosuppressant therapy could slow down the onset of type 1 diabetes mellitus if it is diagnosed early.</paragraph></subquestion><subquestion id="d433d97bb98f4f03ab4b719c603acb70" part_nr="Q" points="0.0" question_nr="14.5"><paragraph id="6671be792ba542ff8b9aee5681fcfe56">The long-acting, subcutaneously administered insulins (e.g. plot C) act longer because their degradation in the circulation is slower.</paragraph></subquestion><pagebreak id="bcfcaf0986b44b558c161b137f346465" /><section id="7e6509e0cebd42179b67ffafc344faf5">Q15</section><paragraph id="4bc197d42b5044edb5749243c5863335">An audiometry exam tests a subject\u2019s ability to hear sounds at different frequencies. Sound waves travel to the inner ear through the ear canal, eardrum, and the bones of the middle ear (air conduction, empty circles (\u25cb) or alternatively, through the bones around and behind the ear (bone conduction, full dots (\u25cf).</paragraph><paragraph id="9d1855e19763438183edbc97584e2e6f">Figure 1. shows the lowest sound intensity (in dB) at a given frequency (Hz) the test subject can still perceive, where a 0 value is the population average and any positive value suggests that a higher than average intensity is required for the subject to hear the sound.</paragraph><figure figid="22" id="f891ce620ecb4a9da97a6b412fa9fcd8" width="1"><caption id="c8ba9bb2dd2f4bd29237385f50d88fd1">Fig.1</caption></figure><subquestion id="9149c917774c45ff96eeb387832a75da" part_nr="Q" points="0.0" question_nr="15.1"><paragraph id="9cf910ef3afb4d1eb1f4356a45720e29">What is the highest frequency at which bone conduction is tested? (Hz)</paragraph></subquestion><paragraph id="8ce58b0d420e42daae375c3e6e0ed9a3">Match the following conditions with the test results in Figure 1 (A-D). Indicate your answer by putting an X in the appropriate box on the answer sheet.</paragraph><subquestion id="ccc2056be89e432ab233ad045488dbdf" part_nr="Q" points="0.0" question_nr="15.2"><paragraph id="39d9e77bfeb144458f7ae5d1300a6530">Old age related reduced sensitivity to high frequency.</paragraph></subquestion><subquestion id="2ffe32383e7f4694806eae149949cb2b" part_nr="Q" points="0.0" question_nr="15.3"><paragraph id="3183aad0f4394c53ad1cddffa10034b6">Middle ear infection impeding the movement of the auditory bones.</paragraph></subquestion><subquestion id="67c27dac7cea4b0f9b96c40aa344f34e" part_nr="Q" points="0.0" question_nr="15.4"><paragraph id="acb4efeed9fd4772994694abb88d168b">Neurological hearing damage causing overall reduction in hearing acuity.</paragraph></subquestion><subquestion id="61d3783c01814d4f8c1fa3d39a8d8682" part_nr="Q" points="0.0" question_nr="15.5"><paragraph id="e3e071a44ac54edab1538d0be44a1c05">Damage caused by an air horn.</paragraph></subquestion><pagebreak id="ce7f9f7454784535b78c007a95877912" /><section id="55d35ab95ea0458e8ddb1ba4b4f16652">Q16</section><paragraph id="516f29ee840f4a539872bbe34b8cc520" /><figure figid="22" id="0b163124ffe74d6e83eb0e465b777186" width="0.8"><caption id="e5b2bfdf9c604ff3807848da33f3a00f">Fig.1. Title 1 = Day 1 Fertilization. Title 2 = Day 2 Morula formation. Title 3 = Day 5 Blastocyst formation. Title 4 = Day 7 Implantation. Title 5 = Day 9 Inner cell mass differentiation. Title 6 = Day 12 Germ disk and then primitive streak formation.</caption></figure><paragraph id="b4a4457c3d144f27ba72cd2e3f28b3e4">In each of the following scenarios (Q.16.1-4), decide if the chorion and the amnion are shared or separate. Indicate your answer by putting an X in the appropriate box (A-D) on the answer sheet.</paragraph><paragraph id="920e836e60f649f4a56b87b6cbd4dd4c">&lt;p&gt;Your options:&lt;/p&gt;

&lt;p&gt;A. Both chorion and amnion are shared&lt;/p&gt;

&lt;p&gt;B. Chorion is separate, amnion is shared&lt;/p&gt;

&lt;p&gt;C. Chorion is shared, amnion is separate&lt;/p&gt;

&lt;p&gt;D. Both chorion and amnion are separate&lt;/p&gt;</paragraph><subquestion id="0934f311b9e04ea2a2590d59de9d65b5" part_nr="Q" points="0.0" question_nr="16.1"><paragraph id="c6136e7c1f3449fd9ac3142f1305ec9b">Dizygotic twins</paragraph></subquestion><subquestion id="b6c5e33f6a674b40b0edcef2369c15d4" part_nr="Q" points="0.0" question_nr="16.2"><paragraph id="b5a6d2fd2c624f65a3c45d4f83dd2b43">Monozygotic twins where splitting occurred in the morula stage</paragraph></subquestion><subquestion id="c65edf5230b646a2807f32a5479ed04c" part_nr="Q" points="0.0" question_nr="16.3"><paragraph id="dc6d8072517146e297403ce9f015ad9b">Monozygotic twins where splitting occurred in the inner cell mass stage</paragraph></subquestion><subquestion id="99017a4f6fbe4ba0bc863f40bc9bb640" part_nr="Q" points="0.0" question_nr="16.4"><paragraph id="94b9c31677d04ffe89fbf74ae8d9bf97">Monozygotic twins where splitting occurred between the formation of the germ disc and the primitive streak</paragraph></subquestion><pagebreak id="1c92b590b02e419082aa93821b7f97b9" /><part id="a7e3bd90b3ed493490b6e2f6de086962">Plant Anatomy and Physiology</part><section id="be1b96ed173f47e9b27156a0f5e681c4">Q17</section><paragraph id="a93d0be5a93547fcbd74f36a3bfb6624">Nutrient availability is thought to be one of the most important conditions influencing stem structure. It is believed to induce changes in the structure of aerenchyma (a spongy tissue of air channels) and the distribution of lacunae (airspaces in aerenchyma tissue). To investigate this, biomechanical parameters of stems of two aquatic plant species (M. s. = M. scorpoides and M. a. = M. aquaticus) were measured under low (&lt;span class="math-tex"&gt;\\(\\downarrow\\)&lt;/span&gt; in table) or high (&lt;span class="math-tex"&gt;\\(\\uparrow\\)&lt;/span&gt; in table) nutrient conditions. Data  were statistically evaluated and the mathematical relationships between these parameters determined as shown in Table1. (ns = non-significant, more stars show greater significant difference).</paragraph><figure figid="22" id="4234ca2716cd4d6f95e475d7cd374058" width="1"><caption id="5e1cc86f021c4ccf84397908f2ec9abd">Table 1. M.s. &lt;span class="math-tex"&gt;\\(\\downarrow\\)&lt;/span&gt; = M. scorpoides in low nutrient conditions. M.s. &lt;span class="math-tex"&gt;\\(\\uparrow\\)&lt;/span&gt; = M. scorpoides in high nutrient conditions. Sign. = Statistical significance. M.a. &lt;span class="math-tex"&gt;\\(\\downarrow\\)&lt;/span&gt; =M. aquaticus in low nutrient conditions. M.a. &lt;span class="math-tex"&gt;\\(\\uparrow\\)&lt;/span&gt; = M. aquaticus in high nutrient conditions. BF = Breaking force.</caption></figure><equation id="288fbb598ec04b67861d814549a06cda">I(M.s.)=\\frac{\\pi R^4}{4}</equation><equation id="a25854f3235445389b51a1075aab9642">I(M.a.)=\\frac{l^4}{12}</equation><equation id="3e042ebb39c84cd9a30675874ebceb18">Icor=I(1-P)</equation><paragraph id="c8f7a12b6bbc40f59b1e658d7a020be6">&lt;p&gt;Where:&lt;/p&gt;

&lt;ul&gt;
\t&lt;li&gt;&lt;strong&gt;Second moment of area&lt;/strong&gt; (I) = describes the geometry of the stem cross section&lt;/li&gt;
\t&lt;li&gt;&lt;strong&gt;Breaking force&lt;/strong&gt; (Fmax)= the maximum force (F) the stem fragment could withstand before breaking&lt;/li&gt;
\t&lt;li&gt;&lt;strong&gt;Porosity &lt;/strong&gt;(P) = the ratio of total lacunar area and total cross-sectional area of stem&lt;/li&gt;
\t&lt;li&gt;&lt;strong&gt;Icor&lt;/strong&gt; = I corrected by porosity&lt;/li&gt;
\t&lt;li&gt;&lt;strong&gt;Radius&lt;/strong&gt; of a circular stem cross section (R)&lt;/li&gt;
\t&lt;li&gt;&lt;strong&gt;Side length&lt;/strong&gt; of a rectangular stem cross section (l)&lt;/li&gt;
&lt;/ul&gt;</paragraph><paragraph id="74c2e791c15a4f299c0628ff7f5c0771">Calculate the mean porosity of the stems of both plant species for both nutrient conditions using two significant figures for your result.</paragraph><subquestion id="54680b8e53d449268eee4bafca8cefb5" part_nr="Q" points="0.0" question_nr="17.1"><paragraph id="551d973049cd4188b816e3507b821537">Mean porosity of M.s. in low nutrient conditions</paragraph></subquestion><subquestion id="e1e3b5cb40c346649c6d816a0adf1033" part_nr="Q" points="0.0" question_nr="17.2"><paragraph id="62876e4cf6f947c4aaacc347433f344a">Mean porosity of M.s. in high nutrient conditions</paragraph></subquestion><subquestion id="c523e802fb8740198b9cf7605cf1170d" part_nr="Q" points="0.0" question_nr="17.3"><paragraph id="5ec573fb16b64f72826db23890d7306f">Mean porosity of M.a. in low nutrient conditions</paragraph></subquestion><subquestion id="9ea30a27a0614a2a9a4274a95420981e" part_nr="Q" points="0.0" question_nr="17.4"><paragraph id="4cba86227cb04b26afbe4b3c8389e0c1">Mean porosity of M.a. in high nutrient conditions</paragraph></subquestion><paragraph id="6c780057fa064dadbb8c46880f6446c3">Representative cross sections of the stems of M.s. and M.a. are shown below.</paragraph><figure figid="22" id="98668d56361c4859a5763d343396562d" width="0.75"><caption id="91e80f1c25254f4fa6cdd75fd1a6961d">Fig.1</caption></figure><subquestion id="38f881ce5f624967851d31c37c42c50d" part_nr="Q" points="0.0" question_nr="17.5"><paragraph id="a2b7f48ffe1342c9a1dcfe47f315b5ce">&lt;p&gt;Match the following cross sections (1 and 2) with the two species (A and B, as specified below). Indicate your answer by putting an X in the appropriate box on the answer sheet.&lt;/p&gt;

&lt;p&gt;A. if the species is M. scorpoides, and&lt;/p&gt;
B. if the species is M. aquaticus.</paragraph></subquestion><subquestion id="6124e6ed6b3e4cecb088e74bab93cae1" part_nr="Q" points="0.0" question_nr="17.6"><paragraph id="5b1a3881ece54660bca96a67fc4f61ef" /></subquestion><pagebreak id="a2f0fc921ec54b71ba3379b505948ba3" /><section id="fd2a086e9d004f12842d3a7311b8d5c6">Q18</section><paragraph id="edcf560e82494ba3a92c2cbde7042b22">Two transcription factors, GNC and GLK, have been implicated in regulating chloroplast development. Three mutants with different combinations of knock-out mutations in these genes were created in &lt;em&gt;A. thaliana&lt;/em&gt; as follows: &lt;em&gt;GNC&lt;/em&gt; or &lt;em&gt;GLK&lt;/em&gt; only and &lt;em&gt;GLC-GNK&lt;/em&gt; with mutations in both genes. These mutants and the wild type plant (WT) were then analysed with respect to various parameters associated with chloroplast development as shown in Figure 1. (different letters indicate significantly different values; NPQ = non-photochemical energy quenching, quantifies the ability of photosystems to dissipate excess excitation energy, higher values indicate greater ability)</paragraph><figure figid="22" id="b7eaf33672914cbfae200e942a41171b" width="0.9"><caption id="27148652b82d4aec9d31df27ca96aac3">Fig.1. i) Average chloroplast fluorescence. ii) Chlorophyll a content (ng/mm2 &lt;span class="math-tex"&gt;\\(\\pm\\)&lt;/span&gt; SD). iii) Chloroplast relative fluorescence intensities. iv) Microscopic images of chloroplasts</caption></figure><subquestion id="cdf87c43b48e4c4db91fe7343f66a21a" part_nr="Q" points="0.0" question_nr="18.1"><paragraph id="e0fafd619f094e2ba4ad225e09f9528f">&lt;p&gt;Select the microscopic techniques which have been used to take the images shown in Figure 1A and Figure 1B. Indicate your answer by putting an X in the appropriate box on the answer sheet.&lt;/p&gt;

&lt;p&gt;\u00a0&lt;/p&gt;</paragraph><paragraph id="0f79d52e33a54bfca643e69849c4faf8">&lt;p&gt;Microscopic techniques:&lt;/p&gt;

&lt;p&gt;A. Bright-field light microscopy&lt;/p&gt;

&lt;p&gt;B. Fluorescence microscopy&lt;/p&gt;

&lt;p&gt;C. X-ray microscopy&lt;/p&gt;

&lt;p&gt;D. Atomic force microscopy&lt;/p&gt;

&lt;p&gt;E. Transmission electron microscopy&lt;/p&gt;

&lt;p&gt;F. Scanning electron microscopy&lt;/p&gt;</paragraph></subquestion><subquestion id="e5c67b779747454d8d30e233d788f40e" part_nr="Q" points="0.0" question_nr="18.2"><paragraph id="286ecef636014e038e1c2055c51fbbdf">Which of the GLK and GNC transcription factor families is the stronger positive regulator of chloroplast development? Indicate your answer using the symbols, &gt; or &lt; or = in the appropriate box on the answer sheet.</paragraph></subquestion><subquestion id="6b75df2015274ecb8d84333ea6accca1" part_nr="Q" points="0.0" question_nr="18.3"><paragraph id="8c76e211a008477eba5f06b8370153e0">&lt;p&gt;Select the single best statement describing the contribution of the GLK and GNC genes to chlorophyll biosynthesis. Indicate your answer by putting an X in the appropriate box on the answer sheet.&lt;/p&gt;

&lt;p&gt;\u00a0&lt;/p&gt;</paragraph><paragraph id="56dd52e96d974f0e9723b28d46ac68fa">A. The GLK and GNC genes modify the expression of chlorophyll biosynthetic genes in separate pathways for chlorophyll biosynthesis.</paragraph><paragraph id="7c32fbfdb8be4298bda9ec95f0b6fc5d">B. GLK genes modify the expression of chlorophyll biosynthetic genes more upstream than the chlorophyll biosynthetic genes controlled by GNC.</paragraph><paragraph id="7b94787d68354fa3a7bf7dc285d1c432">C. GNC genes modify the expression of chlorophyll biosynthetic genes more upstream than the chlorophyll biosynthetic genes controlled by GLK.</paragraph><paragraph id="63da609c48a5478bb68e184eb34f7a2a">D. The relative contribution of the GLK and GNC regulated chlorophyll biosynthetic genes cannot be determined based on the data above.</paragraph></subquestion><pagebreak id="ce16aa93e3d5400d9df72ec408954e83" /><section id="b2b125c1f0c5408b96000246219421ba">Q19</section><paragraph id="93088ef5ade646caac28afb3e37fb0cc">Two transcription factors, GNC and GLK, have been implicated in regulating chloroplast development. Three mutants, GNC single-, GLK single-, and GNC-GLK double mutants were analysed for the expression of nine genes (PSAE-2, PSAF, PSBY,CHLM, LHCA2, LHCB3, rbcL, psaC and psbA) using reverse transcription quantitative PCR (RT-qPCR).</paragraph><figure figid="22" id="0016a385172341e2b0e669e120a5e721" width="0.5"><caption id="d83e209c62c64c0e8fc3b0e30a6e60c3">Fig.1. Axis = Relative expression</caption></figure><subquestion id="e3470bc559e540aeb06a3569ab99bf71" part_nr="Q" points="0.0" question_nr="19.1"><paragraph id="c563737185184f9f8aeb312025242b6c">Arrange the following experimental steps in the appropriate order and fill in the flowchart to summarise the workflow of a RT-qPCR. Put one letter in each box. You do not have to use all the letters. We already put letter A in the correct position for you.</paragraph><list id="bff8122f079743cabec0db58f36af8c0"><item id="01b93aa0271f4959bc9c5ea085a56565">&lt;em&gt;&lt;strong&gt;A. Reverse transcribe into cDNA (complementary DNA) using reverse transcriptase&lt;/strong&gt;&lt;/em&gt;</item><item id="e29ad1142cf44e4da8a0927e4f48e814">B. Add primers of random sequence and DNA-dependent DNA polymerase</item><item id="926dfbf40d3348c3a45f5b9a007cacb4">C. Analyse the results by determining the number of PCR cycles needed for each sample to reach a threshold fluorescence and compare these to one another</item><item id="64684f85710141f6b80787f7012c0070">D. Take plant tissue samples</item><item id="1c3aeac4f12a40f18ec9e3330e1663f1">E. Purify total mRNA</item><item id="28f6ca9c62724615af3ad31803f5b0b2">F. Add a sequence-specific probe containing a quencher and a fluorophore, primers for the sequence of interest and DNA-dependent DNA polymerase</item><item id="8ae2b94698f34b1c9a8a82645f9ec56e">G. Measure the fluorescence intensity at the end of the PCR amplification with a spectrophotometer</item><item id="478d7f0ee1724e3bbdd1dde9693dbee0">H. Purify total DNA with RNase</item><item id="cae4efa65f4a40c3ba63d67e16fd857f">I. Incubate in a thermocycler under the appropriate conditions to allow for the amplification of the sequence of interest and the release of the fluorophore from the probe. Simultaneously, computationally quantify the increase in fluorescence due to the fluorophore release</item><item id="41c30b4348e74de585e9208f7c8c611f">K. Create cell lysate</item></list></subquestion><subquestion id="d7bacc81ff9a49d2b8d44c2be1364cf4" part_nr="Q" points="0.0" question_nr="19.2"><paragraph id="6468620222f94d0187c7a3152270a97f">Given that GNC and GLK act as transcription factors, determine if they are positive or negative regulators of each of the the nine target genes (PSAE-2, PSAF, PSBY,CHLM, LHCA2, LHCB3, rbcL, psaC and psbA). Select the genes that are &lt;u&gt;&lt;strong&gt;upregulated&lt;/strong&gt;&lt;/u&gt; by GNC and GLK and indicate your answer by putting X in the appropriate boxes on the answer sheet.</paragraph></subquestion><pagebreak id="23d0e64beeba4712a2794719167023dc" /><section id="dfc6320fa8764f3999482285d5e9b709">Q20</section><paragraph id="67f374abeae4469281d9555c7d7b123b">The following question concerns the electron transport chain of photosynthesis. Note that NADP+/NADPH is at a higher free energy state than the manganese cluster of the oxygen evolving complex but lower than the central chlorophyll pair of photosystem II in the excited state.</paragraph><subquestion id="0c709b391ae7461f8f60a134f707491a" part_nr="Q" points="0.0" question_nr="20.1"><paragraph id="20a65dcc26f246549c99766117bb964b">&lt;p&gt;Put the following oxidised/reduced pairs in an order of increasing redox potential. Put one letter in each box on your answer sheet. We already put letter F in the correct box.&lt;/p&gt;</paragraph><list id="dfcb4cd4bf604520a083bcd909b33fd9"><item id="49315561a9fe4bd5a64a5cbbd9a318b4">A. Oxidised/reduced central chlorophyll pair of photosystem I in the excited state</item><item id="90ff014f01ec48f39831706f1392e519">B. &lt;span class="math-tex"&gt;\\(NADP^+\\)&lt;/span&gt; or NADPH</item><item id="0eee53da7589400e943dd1426550638e">C. Fully reduced/fully oxidised manganese cluster of the oxygen evolving complex</item><item id="07c7671984a94646b3f15b0561c2f51d">D. Oxidised/reduced central chlorophyll pair of photosystem II in the ground state</item><item id="11eb5e8877244d50970cddff30e02b57">E. Oxidised/reduced central chlorophyll pair of photosystem II in the excited state</item><item id="faaab4ed9ba7498ab7d24f1f7d851175">&lt;em&gt;&lt;strong&gt;F. Oxidised/reduced electron carrier Tyr residue&lt;/strong&gt;&lt;/em&gt;</item></list></subquestion><pagebreak id="248725e5d8384367a070ad5295f98a97" /><section id="c4a3a0c6e3404bf88547c47963c2842e">Q21</section><paragraph id="d59eff4ea7fe47f988621f2c98e283f9">Not all flowers use the ABC model of genetic regulation. Nigella species have spiral flowers and variable numbers of sepals (C), petals (D), stamens (E) and carpels (F) under the control of a genetic system different from that of ABC.</paragraph><paragraph id="ca11733678da4be8b99aab02b99362a9">Fig.1 Genes involved in flowering (leftmost column) were silenced and their morphology was assessed. The number of each organ (C-F) was measured and results are indicated in the bar charts (right column). Bar G shows the sum of all the numbers (C-F).</paragraph><figure figid="22" id="98a6f513ed0f42498f2e5ef95fdc6e2c" width="0.3"><caption id="15d404f961054daeaaed387317e130fa">Fig.1. WT = Wild-type. Legend 1 = Naturally occurring gene 1 KO mutant + genes 6 and 7. Legend 2 = Naturally occurring gene 1 KO mutant + gene 10.</caption></figure><subquestion id="f5405526cea84278b303ded6297c69c9" part_nr="Q" points="0.0" question_nr="21.1"><paragraph id="d946d3cfd6ef4fc0912b6bc76abbb743">Each gene is expressed according to one of the following patterns (darker = more expression). Match the genes (1-13) with the patterns (A-H). Indicate your answer by putting the letters A-H in the appropriate boxes on the answer sheet. Each letter is only used once!</paragraph><figure figid="22" id="5ba0befbc34f4005967c7e46ff76f9e1" width="1" /></subquestion><pagebreak id="27f783290cb74ffaa8d0bbde85355196" /><section id="4d7d21cc538e40f393fb4cc3d3e753fa">Q22</section><paragraph id="f8a717b56e37418f899d99578852024b">Venation and size of leaves are important factors influencing plant physiology. Veins can be categorised hierarchically into four groups: midvein (&lt;span class="math-tex"&gt;\\(1^\\circ\\)&lt;/span&gt;), secondary (&lt;span class="math-tex"&gt;\\(2^\\circ\\)&lt;/span&gt;), tertiary (&lt;span class="math-tex"&gt;\\(3^\\circ\\)&lt;/span&gt;) and quaternary (4\u00b0 or minor veins) as indicated by arrows on Figure 1. Scaling between vein density (= vein length per total leaf area) and leaf area as well as vein diameter and leaf area were determined for 500 species of dicotyledons. [&lt;span class="math-tex"&gt;\\(\\bullet\\)&lt;/span&gt; = data from species with branched veins; &lt;span class="math-tex"&gt;\\(\\circ\\)&lt;/span&gt; = data from species with parallel veins]</paragraph><figure figid="22" id="62ea90a121b448b0aee8f521fc687b08" width="1"><caption id="b94004c3af31454ea6f12e660f4566f8">Fig.1. Axis 1 = Leaf area (&lt;span class="math-tex"&gt;\\(cm^2\\)&lt;/span&gt;). Axis 2 = Vein density (= vein length per leaf area; &lt;span class="math-tex"&gt;\\(cm\\cdot cm^{-2}\\)&lt;/span&gt;). Legend 1 = Midvein. Legend 2 = Minor veins.</caption></figure><paragraph id="2595e203155c4e79ac5dccdd1b1b3338">Based on these data, a model was created (Figure 2) which is generally true for most of the species examined. It details the change of various structural features during the development of a dicotyledonous leaf.</paragraph><figure figid="22" id="7d6bfb9e76354712875e927ae648678d" width="1"><caption id="f4a89403e2094bf3a793df204d7ca5cb">Fig.2. Axis 1 = Days of leaf expansion. Axis 2 = Leaf area (&lt;span class="math-tex"&gt;\\(cm^2\\)&lt;/span&gt;). Axis 3 = Vein development. Axis 4 = Vein density (&lt;span class="math-tex"&gt;\\(mm\\cdot mm^{-2}\\)&lt;/span&gt;). Axis 5 = Vein diameter (mm). Legend 1 = Cell division growth. Legend 2 = Cell expansion growth. Legend 3 = Minor veins.</caption></figure><paragraph id="71e7d33098ef4f82a77efa5f67723b71">Determine the relationship between the following pairs (A and B). Indicate your answer on the answer sheet by using the symbols &gt; or &lt; or =.</paragraph><subquestion id="63cd68eed8604bbfa8f8b56e15c033d0" part_nr="Q" points="0.0" question_nr="22.1"><paragraph id="96cb057c78894651ab7011ea333eaf2c">&lt;p&gt;A. Vein density scaling with leaf surface area for midvein and &lt;span class="math-tex"&gt;\\(2^\\circ\\)&lt;/span&gt;.&lt;/p&gt;

&lt;p&gt;B. Vein density scaling with leaf surface area for &lt;span class="math-tex"&gt;\\(3^\\circ\\)&lt;/span&gt; and &lt;span class="math-tex"&gt;\\(4^\\circ\\)&lt;/span&gt;.&lt;/p&gt;</paragraph></subquestion><subquestion id="dedd7062d9f94283812375776446eb27" part_nr="Q" points="0.0" question_nr="22.2"><paragraph id="74c7187b6ab44bbe841f71d22ac89fb3">&lt;p&gt;A. Vein density scaling with leaf surface area in species with branched veins.&lt;/p&gt;

&lt;p&gt;B. Vein density scaling with leaf surface area in species with parallel veins&lt;/p&gt;</paragraph></subquestion><subquestion id="2880a62039b54eb3ba2f980d948ac368" part_nr="Q" points="0.0" question_nr="22.3"><paragraph id="e735a5d7226a49eaa03a0d185306fa5d">&lt;p&gt;Identify how density of veins of different hierarchical levels change over time and label Fig.2.c below accordingly. Match the letters below (A-D) with the appropriate roman numeral on your answer sheet. Indicate your answer by putting an X in the appropriate box on the answer sheet.&lt;/p&gt;

&lt;p&gt;A. &lt;span class="math-tex"&gt;\\(1^\\circ\\)&lt;/span&gt; veins&lt;/p&gt;

&lt;p&gt;B. &lt;span class="math-tex"&gt;\\(2^\\circ\\)&lt;/span&gt; veins&lt;/p&gt;

&lt;p&gt;C. &lt;span class="math-tex"&gt;\\(3^\\circ\\)&lt;/span&gt;veins&lt;/p&gt;

&lt;p&gt;D. Minor veins&lt;/p&gt;</paragraph><figure figid="22" id="56b37f77e3494c9b9562202fbe8f2f4c" width="0.75"><caption id="ef7aaa4ef2504d67ab6adac7da1ee4a5">Fig.3. Axis 1 = Days of leaf expansion. Axis 2 = Vein density (&lt;span class="math-tex"&gt;\\(mm\\cdot mm^{-2}\\)&lt;/span&gt;)</caption></figure></subquestion></question>
"""

IBO2019_DATA[
    "Morning_2"
] = r"""
<question id="q0" points="0.0"><title id="362a1d5a894042ad936cb6e152bd4b4c">Morning 2nd</title><pagebreak id="6ab5482b71d64cb2a09d77c0c2cef907" /><part id="67bdb87864c24c7c9f0321b9ff92a9a6">Genetics and Evolution</part><section id="5ee38a2b7803452a99b6745b6c63bacc">Q23</section><paragraph id="7c4d01fb03654613a92bdfc9723235c0">The gene that determines red or black fur colour is X chromosome linked. During female embryonic development one of the two parental copies of chromosome X becomes inactivated. If a female cat inherits different alleles from its parents, we can observe the mosaicism. For example, if the maternally derived X chromosome becomes inactivated and the paternally remains active such cells form black patches on the cat's coat. If the paternally derived X chromosome turns inactive and the female derived remains active such cells form red patches. (The white patches originate through a different mechanism.)</paragraph><figure figid="22" id="47d49269bbea4aae8afeea7a4cfab310" width="0.5"><caption id="87bd9f50624c432ea06dbc39bc3b0cec">Fig.1</caption></figure><paragraph id="d2805748d6964554b53883af16aa9945">&lt;p&gt;You have a list of possible observations regarding the inheritance of red/black coat colour:&lt;/p&gt;

&lt;p&gt;A. There are no smaller red spots in the middle of the large black spots or vice versa.&lt;/p&gt;

&lt;p&gt;B. Black spots are biased towards the back of the animals&lt;/p&gt;

&lt;p&gt;C. The mosaic spots are large.&lt;/p&gt;

&lt;p&gt;D. The red/black spot boundaries are sharp.&lt;/p&gt;

&lt;p&gt;E. The feet of the cats are usually white.&lt;/p&gt;

&lt;p&gt;F. The distribution of spots in different cats is very variable.&lt;/p&gt;</paragraph><paragraph id="4b82f7050acd442abc549967b5b734c5">Match the observations with one of the four conclusions listed in the table with an X. Only one letter should be matched with one conclusion and one letter may only be used once.</paragraph><subquestion id="d2aa151eccdb4f3caf078b2d1330f19b" part_nr="Q" points="0.0" question_nr="23.1"><paragraph id="a7d88070cb4245528fcf726e0d962fd7">X chromosome inactivation takes place during early embryogenesis.</paragraph></subquestion><subquestion id="15bf936eb8ac4326a206dc6fd1e679b1" part_nr="Q" points="0.0" question_nr="23.2"><paragraph id="55485e4401914f2a89abcec146e4e16d">Inactivation of the maternal or paternal X chromosome is a random event.</paragraph></subquestion><subquestion id="c0e4e18aac3f4e3392b8bf40583dd3bb" part_nr="Q" points="0.0" question_nr="23.3"><paragraph id="15bfbd1177c04c72bb5ca5857796f759">X chromosome inactivation is irreversible.</paragraph></subquestion><subquestion id="db12c4cb7c794dbcbe93b2c50237d69c" part_nr="Q" points="0.0" question_nr="23.4"><paragraph id="9d79f883d756410c9fde6d99de9463bd">X chromosome inactivation is a cell autonomous process, i.e. the neighboring cells do not influence it.</paragraph></subquestion><pagebreak id="ad1aaee318384b1c960944b488f64641" /><section id="0916bbe55d44482aae6811136a5bee6b">Q24</section><paragraph id="1ed12f3bb6c940bcb36fdaa05b942d56">Carolyn does not know who the father of her daughter Lucy is. She knows that the father is one of the five members of the Happening rock band. You collected DNA samples from the seven people listed in the table and analyzed five (A through E) of the VNTR sequences. (The VNTRs - variable number tandem repeats - are DNA sequences present in variable number of tandemly arranged copies in both autosomes and sex-chromosomes) The figures in the table represent the number of copies in the different VNTRs.</paragraph><figure figid="22" id="1b7129a3f9624ee5acbb08db3b6f3389" width="1"><caption id="ec3e050ac2b2456aa383141a96645b2d">Table 1</caption></figure><subquestion id="ee537a4c45704e6a8fb6a67bc65df453" part_nr="Q" points="0.0" question_nr="24.1"><paragraph id="f31b897cf1cf42759a2e149bc8905e10">&lt;p&gt;Indicate with an X who Lucy\u2019s father is.&lt;/p&gt;\r
\r
&lt;p&gt;A. Frank&lt;/p&gt;\r
\r
&lt;p&gt;B. Conard&lt;/p&gt;\r
\r
&lt;p&gt;C.\u00a0Andrew&lt;/p&gt;\r
\r
&lt;p&gt;D. Bill&lt;/p&gt;\r
\r
&lt;p&gt;E. Lewis&lt;/p&gt;</paragraph></subquestion><subquestion id="cff3e1b6eeba429dae260ea80a37f5be" part_nr="Q" points="0.0" question_nr="24.2"><paragraph id="668eb75ce7bc49c18dd0fc882ebdd1bf">Carolyn and Frank have a son called Rupert. List all possible VNTRs copy numbers Rupert can have at the analysed VNTR sequences (A-E). Treat the rows independently from one another. If the two alleles of a specific VNTR sequence has different number of copies, indicate it with \u2018+\u2019 sign between them.</paragraph><figure figid="22" id="5ab8004d96134a2994a7fd2d763d10dc" width="1" /></subquestion><pagebreak id="f0c70e8ee7af4d8ca1f84b7bf6007b93" /><section id="309fbee2b6504d7eb92e0c35fdeeffdb">Q25</section><paragraph id="85d87c159cbc4ebaae3a669c3029ebeb">The &lt;em&gt;Eco&lt;/em&gt;RI restriction enzyme recognizes the 5\u2019G*AATTC3\u2019 nucleotide sequence and cuts the DNA at the \u201c*\u201d site. You have a plasmid preparation with billions of copies of a plasmid composed of &lt;span class="math-tex"&gt;\\(1.3\\times10^4\\)&lt;/span&gt; base pairs. Let's assume that the sequence of base pairs is random in the plasmid DNA.</paragraph><subquestion id="53bff28890754edc88030e7bcf8fc7fd" part_nr="Q" points="0.0" question_nr="25.1"><paragraph id="73645ddcf4cb494fb48d1797d971be3e">On average, how many DNA fragments form following the digestion of the plasmid DNA by the &lt;em&gt;Eco&lt;/em&gt;RI restriction enzyme?</paragraph></subquestion><pagebreak id="97cbaff631c34f4185f493afd6668b1d" /><section id="6ba39d93712f41579a7c7cef68f0a329">Q26</section><paragraph id="36b89929870a4e40806493a4d98be3d4">A single base pair deletion mutation occurred in a short gene, which changes the MATE peptide's genetic code into MATEK. The peptide sequence is given based on the one letter codon table. You can find the original peptide and nucleotide sequence below, but some nucleotides are missing.</paragraph><figure figid="22" id="d9bef76716a145ea99025b811e6c8815" width="1"><caption id="e28081b84a73405f90ed7bad9f83c0f4">Fig.1. Original sequence. PS = Peptide sequence. NS = Nucleotide sequence.</caption></figure><subquestion id="544a9dd3b47342599d1d59d05510f5c5" part_nr="Q" points="0.0" question_nr="26.1"><paragraph id="6d0ae97c7b464aa88f221c72b167f654">Give the letter of the missing nucleotides (1-5). If in one position more than one type of nucleotide can occur, give only one of them.</paragraph></subquestion><pagebreak id="6639ae49d2e44791844a7e034a20f9b5" /><section id="13b00e1d75bd4ffab290303eec805edb">Q27</section><paragraph id="b5485215da2741e2a97e9ec5a0e494a7">&lt;p&gt;Karolyn and Fred visit your genetic counselling office. They tell you the data summarized in the table. You aware of the fact that:&lt;/p&gt;

&lt;p&gt;(i) the absence of patella is caused by the dominant mutation Np (Nail-patella).&lt;/p&gt;

&lt;p&gt;(ii) The Np mutant allele (and of course the Np+ normal gene) and the AB0 blood group genes (IA, IB and i respectively) are inherited on chromosome 9 and are 18 cM apart.&lt;/p&gt;</paragraph><table columns="|l|c|c|c|" id="392d5ac543334d6e9b064f5288067aea" top_line="1"><row bottom_line="1" id="ee5a209c9e04469291f455778bfe6af9" multiplier="1"><cell id="3632e194297641f8baaae23270f99cf8">Person</cell><cell id="61d844f4cb054324bef3166787a3f920">Does she/he have patella?</cell><cell id="81540692d999413e96bebb1ab06db859">Blood group</cell></row><row bottom_line="1" id="0b4894e17cf54e6d906a6e8597854c04" multiplier="1"><cell id="2ead6fb47701470ab810350d87ed8ce1">Karolyn</cell><cell id="88d0802a730a42cf86a00a035cae0b81">No</cell><cell id="d2e32980ea3d41898366396ea7a431e7">B</cell></row><row bottom_line="1" id="b5f96afc9b7a4894a24c157f64403c32" multiplier="1"><cell id="b569091e8f1a4cc4a65e51988571be3e">Karolyn's mother</cell><cell id="2fbd8b68a022424d964d3771f6a2dff8">Yes</cell><cell id="06341342a63e44f3910e3821efbc51aa">B</cell></row><row bottom_line="1" id="9c480f3ac3204fd1bcf63f5d5dcc5103" multiplier="1"><cell id="b0bcdce87a2e4c4288cc72a83b6005f2">Karolyn's father</cell><cell id="dc1a0f3954ff43edae1b4c4053432a80">No</cell><cell id="d8564b0adf1d476f88cfbdd2503deb16">0</cell></row><row bottom_line="1" id="39f7bdad8af84209886e0c630d8c7752" multiplier="1"><cell id="d0f409b9a3ff4bfa9d5a9184c37b614a">Fred</cell><cell id="7dfa1270737c4375a600f367511c32ce">Yes</cell><cell id="bf103f5ae99f4aafa7c028f5d66b6165">0</cell></row><row bottom_line="1" id="5935e7a14c844d22a78383222510aec7" multiplier="1"><cell id="3aa7eb50a0504008b8096b2794a38d7e">Fred's mother</cell><cell id="b7fbdaee0e7d4cbdacb7d544366c62f0">Yes</cell><cell id="2c41751c0e564ed1b7c0b48a183f1a6c">A</cell></row><row bottom_line="1" id="cdb039f5d6cf4889b7cef97bd064ed57" multiplier="1"><cell id="b8b215f0df4540b8a52a5284f00ec6b4">Fred's father</cell><cell id="323be2ac85f64f3ab20a9e4a3da47734">Yes</cell><cell id="06a1ca8dd314497d90c0691cbfb5a7ad">0</cell></row></table><paragraph id="4b88acd42c1540149cc3d514b1176b0f">Determine Karolyn's and Fred's genotypes. The alleles on one side of \u2018/\u2019 sign are linked. (For example, in case of &lt;span class="math-tex"&gt;\\(Np\\:i\\)&lt;/span&gt; / &lt;span class="math-tex"&gt;\\(Np^+\\:I^A\\)&lt;/span&gt; genotype, &lt;span class="math-tex"&gt;\\(Np\\)&lt;/span&gt; and &lt;span class="math-tex"&gt;\\(i\\)&lt;/span&gt; alleles are linked and &lt;span class="math-tex"&gt;\\(Np^+\\)&lt;/span&gt; and &lt;span class="math-tex"&gt;\\(I^A\\)&lt;/span&gt; alleles are linked.)</paragraph><subquestion id="891492fc3b174b658c0070df7fbbfa75" part_nr="Q" points="0.0" question_nr="27.1"><paragraph id="0883451689234cb29b0098110286a818">&lt;p&gt;What is Karolyn's genotype? Indicate your answer with an X.&lt;/p&gt;

&lt;p&gt;A. &lt;span class="math-tex"&gt;\\(Np\\:I^B\\)&lt;/span&gt; / &lt;span class="math-tex"&gt;\\(Np\\:I^B\\)&lt;/span&gt;&lt;/p&gt;

&lt;p&gt;B. &lt;span class="math-tex"&gt;\\(Np\\:I^B\\)&lt;/span&gt; / &lt;span class="math-tex"&gt;\\(Np+\\:i\\)&lt;/span&gt;&lt;/p&gt;

&lt;p&gt;C. &lt;span class="math-tex"&gt;\\(Np\\:I^B\\)&lt;/span&gt; / &lt;span class="math-tex"&gt;\\(Np\\:i\\)&lt;/span&gt;&lt;/p&gt;

&lt;p&gt;D. &lt;span class="math-tex"&gt;\\(Np+\\:I^B\\)&lt;/span&gt; / &lt;span class="math-tex"&gt;\\(Np\\:i\\)&lt;/span&gt;&lt;/p&gt;

&lt;p&gt;E. &lt;span class="math-tex"&gt;\\(Np+\\:i\\)&lt;/span&gt; / &lt;span class="math-tex"&gt;\\(Np\\:I^B\\)&lt;/span&gt;&lt;/p&gt;</paragraph></subquestion><subquestion id="875094ead06b403283f9f82cc91fffb0" part_nr="Q" points="0.0" question_nr="27.2"><paragraph id="766cae8736a44a1f9955d840bd780db5">&lt;p&gt;What is Fred's genotype? Indicate your answer with an X.&lt;/p&gt;

&lt;p&gt;A. &lt;span class="math-tex"&gt;\\(Np\\:i\\)&lt;/span&gt; / &lt;span class="math-tex"&gt;\\(Np\\:i\\)&lt;/span&gt;&lt;/p&gt;

&lt;p&gt;B. &lt;span class="math-tex"&gt;\\(Np+\\:i\\)&lt;/span&gt; / &lt;span class="math-tex"&gt;\\(Np+\\:i\\)&lt;/span&gt;&lt;/p&gt;

&lt;p&gt;C. &lt;span class="math-tex"&gt;\\(Np+\\:i\\)&lt;/span&gt; / &lt;span class="math-tex"&gt;\\(Np+\\:I^A\\)&lt;/span&gt;&lt;/p&gt;

&lt;p&gt;D. &lt;span class="math-tex"&gt;\\(Np\\:i\\)&lt;/span&gt; / &lt;span class="math-tex"&gt;\\(Np+\\:I^A\\)&lt;/span&gt;&lt;/p&gt;

&lt;p&gt;E. &lt;span class="math-tex"&gt;\\(Np+\\:I^A\\)&lt;/span&gt; / &lt;span class="math-tex"&gt;\\(Np+\\:I^A\\)&lt;/span&gt;&lt;/p&gt;</paragraph></subquestion><subquestion id="cbe0f041cc2546acabe0b6d1185bcb96" part_nr="Q" points="0.0" question_nr="27.3"><paragraph id="180136bc168d4ccf8a8cf6d5eaa39287">Karolyn is pregnant. Her foetus has blood group B. Add the probability in percentage that her child will have no patella.</paragraph></subquestion></question>
"""

IBO2019_DATA[
    "Experiment_1"
] = r"""
    <question id="q0" points="0.0"><title id="08e806a2d02e4891872c4aa8b24a7ecc">Experiment 4</title><part id="df24b80b8be04a02b84013bfdea3f4c6">SUBTASK 1. PRACTICAL TASK IN BIOINFORMATICS</part><paragraph id="e69bab1584af4121ae17cf5cd9c15aec">&lt;p&gt;Modern biological research in the 21st century is inconceivable without bioinformatic analyses.  In this practical task, we present an example of how bioinformatic data are processed in order to investigate evolutionary issues of bacterial resistance and susceptibility to antibiotics.&lt;/p&gt;</paragraph><paragraph id="eb120a9938954356abe5c006db0b2a30">&lt;p&gt;Please do not start using the Geneious Prime software until you have completely read and understood the instructions. Otherwise, you may cause changes to the data that cannot be restored. The lab assistant will not assist you in replacing lost or corrupted data, so you should proceed step by step only.&lt;/p&gt;</paragraph><box id="b256ac6d6ac045268a82d725b4300841"><paragraph id="29ad8d51f52542feaf8d1ae4aafc3c40">&lt;p&gt;&lt;strong&gt;IMPORTANT TECHNICAL NOTES ON CHEATING&lt;/strong&gt;&lt;/p&gt;

&lt;ul&gt;
\t&lt;li&gt;&lt;strong&gt;During the practical, you will use the Geneious Prime software that is already running on your computer.&lt;/strong&gt;&lt;/li&gt;
\t&lt;li&gt;&lt;strong&gt;&lt;u&gt;Please do not use or even open any other software&lt;/u&gt;, except for this Geneious software and the Neurophysiology software that you will use for the other practical that is also already running.&lt;/strong&gt;&lt;/li&gt;
\t&lt;li&gt;&lt;strong&gt;Your computer records all your activities, so if you open any other software, or if you connect your computer to the internet, you will be excluded from this task and lose any points that you scored here.&lt;/strong&gt;&lt;/li&gt;
&lt;/ul&gt;</paragraph></box><paragraph id="82aa7d0970dd426fa06e94502202f457">&lt;p&gt;Chloramphenicol acetyltransferase (CAT) is responsible for chloramphenicol resistance in &lt;em&gt;Escherichia coli&lt;/em&gt;..The CAT enzyme is made up of 219 amino acids and encoded by a gene consisting of 660 nucleotides It is a well-known phenomenon that antibiotic resistance spreads rapidly across pathogens. This is why many scientific studies are looking at ways to prevent this. One approach is to explore whether the inverse of resistance, that is, recovered antibiotic susceptibility, can also be passed on.&lt;/p&gt;</paragraph><paragraph id="e161ac6fcaba43389d13b1a4742677ed">&lt;p&gt;In this task, you will be required to analyse the results of such a model experiment. An &lt;em&gt;E. coli&lt;/em&gt; strain originally resistant to chloramphenicol was exposed to periodic heat stress, and strains were isolated after each stress plus inoculation cycle. Some of these strains remained resistant, while others have lost their defences against the antibiotic.&lt;/p&gt;

&lt;p&gt;Freshly inoculated liquid cultures of 16 strains from the experiment were grown with chloramphenicol for 2 days. 10 mL of each culture was diluted by 100X. Then we determined the population density of each culture and measuring optical density at 600nm with a spectrophotometer. This value is directly proportional to the population density. Table 1 shows &lt;span class="math-tex"&gt;\\(OD_{600}\\)&lt;/span&gt; information of the 16 strains.&lt;/p&gt;</paragraph><figure figid="22" id="61b162ca785a4b56bc433d98f2e48ea0" width="0.6" /><subquestion id="3937081333724e9a876d5e5c917f7318" part_nr="Q" points="0.0" question_nr="4.1.1"><paragraph id="5a8625ba2ff8486a85ee5ea774b19389">&lt;strong&gt;Study Table 1 carefully and identify which strain(s) lost its/their resistance. Indicate your answer by putting an X in the appropriate box on the answer sheet.&lt;/strong&gt;</paragraph></subquestion><paragraph id="8098e16263bf48818f173cce80d0a4be">&lt;p&gt;In order to examine whether strains that became susceptible have evolved independently from each other or they have a genetic relationship, we will analyse the sequence of a segment of their CAT gene. Since CAT is an enzyme, it can be assumed that its loss of function can be primarily caused by mutations in the active site of CAT.&lt;/p&gt;

&lt;ul&gt;
\t&lt;li&gt;Log in to your computer and click on the Geneious Prime icon in the bottom bar:&lt;/li&gt;
&lt;/ul&gt;</paragraph><figure figid="22" id="5e2da94034e14718bcfe59afb73fe090" width="0.05" /><paragraph id="6393f7a619f14077b9a52db980d4a18f">&lt;ul&gt;
\t&lt;li&gt;cDNA sequence segments of mRNAs from the CAT genes of the examined 16 strains have previously been uploaded.&lt;/li&gt;
\t&lt;li&gt;You can display them on your screen by clicking on the \u2018+\u2019 sign next to the folder entitled \u2018IBO2019\u2019 in the left pane, which will show the folders of this subtask. Clicking on \u2018Subtask1\u2019 will display the sequences.&lt;/li&gt;
\t&lt;li&gt;Your first task will be to compare the sequences.&lt;/li&gt;
\t&lt;li&gt;By ticking the boxes to the left of the sequence names, select all 16 sequences, then click on \u2018Multiple Align\u2019 command in the \u2018Align/Assemble\u2019 drop-down menu. In the pop-up window, do not change any settings, and click on OK. The alignment of the sequences can take up to 1 minute, so be patient.&lt;/li&gt;
\t&lt;li&gt;When the alignment process is completed, the progress bar will disappear and an additional line will appear under the original sequences. Double-click on this new line. You should see a lot of grey bars with some scattered tiny coloured spots on them.&lt;/li&gt;
\t&lt;li&gt;Zoom the sequence segments with the Ctrl + mouse scroll key combination. The letter identifying the bases will appear. Above the sequences, the consensus sequence is shown which appears to be the most common base sequence during the alignment of the examined instances.&lt;/li&gt;
\t&lt;li&gt;Use the scroll button of the mouse to move left and right along the sequences.&lt;/li&gt;
&lt;/ul&gt;</paragraph><paragraph id="4934d3f3898a42c88ddd5f07dd5159cb">&lt;p&gt;Answer the following questions in the answer sheet by writing your answer in the appropriate boxes.&lt;/p&gt;</paragraph><subquestion id="f37af37f8f2643cf9a1bcd88a6316c6c" part_nr="Q" points="0.0" question_nr="4.1.2.a"><paragraph id="1aa8aeedbd7242c89dd431c2bf7e0100">Specify the position (ordinal number of the base) in sequence segment 250-350 in which Strain1 has a base different from the consensus sequence.</paragraph></subquestion><subquestion id="bacc43a01829455396b04b279e222cb3" part_nr="Q" points="0.0" question_nr="4.1.2.b"><paragraph id="df04e83378164f28ab14de9b377a361f">Specify the base in Strain1 at the position identified in Q4.1.2.a by writing the one-letter code of the base in the appropriate box.</paragraph></subquestion><subquestion id="9a8ce71ae9f2485b9553597088dd9ddf" part_nr="Q" points="0.0" question_nr="4.1.2.c"><paragraph id="81777970b3104bd8b32165bfdfecc96b">Find the position(s) where the most differences from the consensus sequence is/are. Specify the ordinal number of this/these position(s).</paragraph></subquestion><subquestion id="b7f8c8f832a247a097ea5e6a45c475dd" part_nr="Q" points="0.0" question_nr="4.1.2.d"><paragraph id="8db71d63c502453d8f76cca8ba56aa6a">Specify the ordinal number of strains that are likely to carry a deletion mutation in the examined gene segment.</paragraph></subquestion><subquestion id="ad21b616c5d0452f90d3eb834621a954" part_nr="Q" points="0.0" question_nr="4.1.2.e"><paragraph id="648c7cd7c0fe40e397bc932018f6523d">Specify the positions of adjacent double base changes, wherein both bases were replaced by their complementary ones.</paragraph></subquestion><subquestion id="d081f50ffb5d4f62ab97cb3a40f5fd8f" part_nr="Q" points="0.0" question_nr="4.1.2.f"><paragraph id="3bb7e3db006a481ebcb3b04ff45be24a">&lt;p&gt;Specify the ordinal number of positions in the sequence where at least four of the strains specified in your response to question Q4.1.1 have the same changes. Count double base changes as two.&lt;/p&gt;</paragraph></subquestion><paragraph id="edfbc6f2c4794e008d9fb5821fcacb6a">&lt;p&gt;Your answer to question Q4.1.2.f. is relevant because all the mutations in question can form the basis for the loss of resistance. Your next task will be to check which mutation causes amino acid substitution in the CAT enzyme.&lt;/p&gt;</paragraph><paragraph id="379164120c6047e38664a86c0c9813f3">&lt;ul&gt;
\t&lt;li&gt;Close the window where you have worked before (NOT the GENEIOUS program, this window only) by clicking on the \u2018X\u2019 button in the upper right corner.&lt;/li&gt;
\t&lt;li&gt;Select the 16 DNA sequences by ticking the boxes to their left. Click on the \u2018Sequence/Translate\u2026\u2019 command in the menu to see the amino acid sequences in the image that appears at the bottom or in a separate window. The consensus amino acid sequence is shown in the top line. Similarly to the DNA sequence, the image can be zoomed using the Ctrl+scroll key combination.&lt;/li&gt;
\t&lt;li&gt;Note that if you zoom into the image to a high magnification, one-letter amino acid codes will be replaced by three-letter codes. In spite of this, ALWAYS USE THE ONE-LETTER AMINO ACID CODES when providing your answers.&lt;/li&gt;
\t&lt;li&gt;Use the scroll button of the mouse without pressing Ctrl to move left and right along the sequences.&lt;/li&gt;
&lt;/ul&gt;</paragraph><paragraph id="2de70aa7f5f64412b6ab6ae5e9e7e3c9">&lt;p&gt;Answer the following questions on the answer sheet.&lt;/p&gt;</paragraph><subquestion id="1aa637595a3f4949af6f0d17069ca555" part_nr="Q" points="0.0" question_nr="4.1.3.a"><paragraph id="3e5f0f62d4e74112aeeb0ce06d2a9561">If a mature bacterial peptide consists of N amino acids, what is the minimum number of nucleotides this peptide can be encoded by? Consider only the coding region of the peptide\u2019s gene! Provide your answer as a formula containing N: (e.g. \u20185N-23')</paragraph></subquestion><subquestion id="3489d7d8af434f43bdb1a6d1a846d33a" part_nr="Q" points="0.0" question_nr="4.1.3.b"><paragraph id="36ab1abdf4024f40902abf6435c355fe">Let us suppose that you want to know which amino acid is encoded by the bases 121\u2013123 of the mRNA from the encoding region of the peptide gene in question Q4.1.3.a. Specify the ordinal number of amino acids in the amino acid sequence of the peptide that need to be checked in order to answer the question.</paragraph></subquestion><paragraph id="1faeae916b2e43aa82e64be6139a6c92">&lt;p&gt;Based on your answers to questions Q4.1.3.a and Q4.1.3.b., check the amino acid positions corresponding to the mutations found in question Q4.1.2.f.&lt;/p&gt;</paragraph><subquestion id="95fb6c22887a4b79afaf89d9bb8c691b" part_nr="Q" points="0.0" question_nr="4.1.3.c"><paragraph id="9c577c059de6437a82daac1ab03c16e4">Specify the ordinal number of the amino acid(s) that changed due to mutations detected in question Q4.1.2.f.</paragraph></subquestion><subquestion id="4204afaaa948451db19adba5b53f7110" part_nr="Q" points="0.0" question_nr="4.1.3.d"><paragraph id="96af6fc08e264554b5eed9973344a174">Specify the ordinal number of the amino acid(s) in which the mutation(s) detected in question Q4.1.2.f. remain(s) silent.</paragraph></subquestion><subquestion id="d20e967cf67c49b99af3664b86f62b73" part_nr="Q" points="0.0" question_nr="4.1.3.e"><paragraph id="6558f39a272448838f8c027d5b7f64ab">Which is the amino acid substitution that appears in all of the chloramphenicol sensitive strains detected in question Q4.1.1.? Specify the ordinal number of that amino acid.</paragraph></subquestion><paragraph id="3e57112d67b94159992b40c66cd3c4b2">&lt;p&gt;We have found the cause of susceptibility above, that is, which amino acid substitution is likely to cause susceptibility. Since the examined DNA segment has changed several times and in several loci during the microevolution of the strains, it is also possible to build a DNA-based phylogenetic tree of the 16 strains. You have to do this with the help of Geneious Prime, as specified below.&lt;/p&gt;</paragraph><paragraph id="aec6819aed9647fcbc20051192920ab7">&lt;ul&gt;
\t&lt;li&gt;If you performed the previous examinations in a new window, close this window (NOT the entire Geneious Prime program!) or if the amino acid sequence appeared in the split bottom pane of the screen, untick the box next to \u2018Translation of 16 sequences\u2019 in the top pane.&lt;/li&gt;
\t&lt;li&gt;If you can only see the DNA segments again, tick/select the tested 16 DNA sequences, then click on the 'Tree' icon in the top menu bar. In the pop-up window, accept the default settings by clicking on 'OK'. It may take a few seconds for the software to generate the phylogenetic tree. If it appears below only, click the upper right corner of the area displaying the phylogenetic tree to enlarge the image so that you can see the window in full screen.&lt;/li&gt;
\t&lt;li&gt;The obtained tree is unstructured (unrooted), so you need to transform it so that it can serve as a basis for conclusions. The first step is to select the oldest (most divergent) strain, which will be the root, also known as the outgroup, of the structured tree.&lt;/li&gt;
\t&lt;li&gt;Since the original phylogenetic tree does not provide accurate information about the oldest strain, you will have to find it in a different way. Click on the 'Distances' tab in the second textual menu bar in the upper left. Thus, a distance matrix is obtained that shows the degree of divergence between each pair of DNA sequences. The larger this number is, the greater the difference in the base sequence of the corresponding DNA segments of the two examined strains is.&lt;br/&gt;
\tIn the upper middle menu, change 'Styles' to 'Heatmap and Numbers'. The darker a rectangle is in the table, the greater is the degree of match between the examined DNA sequences of the two strains. Select the strain with the darkest rectangles. The sequence of this strain shows the greatest degree of match with the sequences of the other strains, i.e. presumably this is the oldest sequence. If you are unsure about your choice, you should add up the number of differences for each \u2018suspicious\u2019 strain per column and select the column with the smallest sum.&lt;/li&gt;
&lt;/ul&gt;</paragraph><subquestion id="5b3d9ac36f594cd2a2130ed46652809f" part_nr="Q" points="0.0" question_nr="4.1.4.a"><paragraph id="6286f2755ba6446d9f38acd9b2672990">Specify the ordinal number of the strain that probably has the oldest sequence among the 16 strains. (For example, if your answer is Strain-23, write \u201823\u2019 on answer sheet.)</paragraph></subquestion><paragraph id="c1c4ec35e9214ad1ae48e49890885c9c">Based on the above data, you can already start structuring the phylogenetic tree.</paragraph><paragraph id="c35cac191fe5451c94ac1530a0ca4267">&lt;ul&gt;
\t&lt;li&gt;To do this, switch back to the 'Tree view' view by clicking on the appropriate tab in the second menu bar from the top.&lt;/li&gt;
\t&lt;li&gt;Subsequently, click on the endpoint nearest to the name of the strain identified in question Q4.1.4.a., and then click on the 'Root' button in the third menu bar that is activated at this time. Thus, an already structured tree is obtained where the common ancestor is the strain identified in question Q4.1.4.a.&lt;/li&gt;
\t&lt;li&gt;For a simpler view, tick the 'Transform branches' box in the right menu bar (Formatting submenu)&lt;/li&gt;
\t&lt;li&gt;Change the scaling from \u201cEqual\u201d to \u201cCladogram\u201d using the scroll down menu just below.&lt;/li&gt;
&lt;/ul&gt;</paragraph><subquestion id="d83f73c104ee45b28ed7c9c454701e46" part_nr="Q" points="0.0" question_nr="4.1.4.b"><paragraph id="7c95dc54a99d4879ad03d16807ea2117">Find the branch containing the sensitive mutants and specify the ordinal number of the resistant strain(s) that is most closely related to these strains. (For example, if your answer is Strain-23, write \u201823\u2019 on the answer sheet .)</paragraph></subquestion><subquestion id="97ddc7763bfc40faa2dcd9eaafc1290f" part_nr="Q" points="0.0" question_nr="4.1.4.c"><paragraph id="ecc5a29c5d864eb1bbfa8852e2a93986">Specify the ordinal number of the two sensitive strains that may have evolved most recently. (For example, if your answer is Strain-23, write \u201823\u2019 on the answer sheet .)</paragraph></subquestion><subquestion id="f6b0a71bf552406e96c20ac6d1427597" part_nr="Q" points="0.0" question_nr="4.1.4.d"><paragraph id="a92cac0f95ed4277adb5d5b3896ef850">&lt;p&gt;For each of the following groups of strains, specify whether it is monophyletic (A), paraphyletic (B), or polyphyletic (C). Indicate your answer by putting an X in the appropriate box on the answer sheet.&lt;/p&gt;</paragraph><paragraph id="b078331150b44d60a7d76f4428da3294">Groups:</paragraph><list id="f7e803a0d362490fbec5eb8aa5db8ec9" label="&quot; &quot;"><item id="d92debcc45e64ca7bad232b0a41c3076" label="\\hspace{1cm}">I. {8; 12}</item><item id="7097fe0a387f4326b51cd8a6f92be224" label="\\hspace{1cm}">II. {5; 6; 16}</item><item id="860f24353b3e47ad8271cb57eed670d7" label="\\hspace{1cm}">III. {3; 7; 10}</item></list><paragraph id="8094d31c79054417b09f905f2fdca4f0">&lt;p&gt;Options:&lt;/p&gt;</paragraph><list id="4a2e87b735f94e1eba91fbb2d301f419" label="&quot; &quot;"><item id="f87e1dc6003b431d8318f43ccfb5e8e4" label="\\hspace{1cm}">A \u2013 monophyletic</item><item id="f7ce4a9d0742498a9571c9f867d3f5a4" label="\\hspace{1cm}">B \u2013 paraphyletic</item><item id="4525fd64c2bf4a4faa65313567db68b6" label="\\hspace{1cm}">C \u2013 polyphyletic</item></list></subquestion><subquestion id="6fe097b97e324c52a0677af410960ae9" part_nr="Q" points="0.0" question_nr="4.1.4.e"><paragraph id="7399e373dcc2479aae08cc9b1c0e4a47">&lt;p&gt;Based on the phylogenetic tree obtained above, you can find autapomorphic, synapomorphic, and plesiomorphic mutations in certain nucleotide positions in the sequences examined in the first part.&lt;/p&gt;

&lt;p&gt;Return to the 16-line DNA sequence alignment obtained in the first part by double-clicking on the \u2018Nucleotide alignment\u2019 line and specify the types of mutations in the following positions from the perspective of the phylogenetic tree. Indicate your answer by putting an X in the appropriate box on the answer sheet.&lt;/p&gt;</paragraph><paragraph id="6b64992272564c728f2a03f05f34368f">Positions:</paragraph><list id="4f73ccbc552f493dae189740bbfdb113" label="&quot; &quot;"><item id="0c9d1a858a4c400b9d884f535fd39beb" label="\\hspace{1cm}">51</item><item id="d4fe32ef9061421d83fc584a4d4f6aa6" label="\\hspace{1cm}">144</item><item id="1a2521132c0c4c9cbd973f7aef977be7" label="\\hspace{1cm}">312</item><item id="7fe2956feaca4f72a9df3f32de0526fb" label="\\hspace{1cm}">447</item><item id="cc455bd834634d4d941a3321ad094796" label="\\hspace{1cm}">531</item></list><paragraph id="cb4f204926bc44b986423b61da6e8230">&lt;p&gt;Options:&lt;/p&gt;</paragraph><list id="24e3c3e146674d8e982f546e386d2701" label="&quot; &quot;"><item id="b404a2db905f47e48d0f3ad667eba058" label="\\hspace{1cm}">&lt;p&gt;A \u2013 autapomorphic&lt;/p&gt;</item><item id="98d963d0d7f84de480429abb0820c46d" label="\\hspace{1cm}">&lt;p&gt;B \u2013 plesiomorphic&lt;/p&gt;</item><item id="fd8aef2e58de4210b771a148e2c8aed4" label="\\hspace{1cm}">&lt;p&gt;C \u2013 synapomorphic&lt;/p&gt;</item></list></subquestion><pagebreak id="68ad38dadcb34b298cc3aabbbf1bfcf8" /><part id="2c13a1a0c61647719f4ce7238f26ed70">SUBTASK 2. EXAMINATION OF NEURONAL ACTIVITY PATTERN ON MOVING ANIMALS</part><box id="8ef16742960c4917a078310d4baed23a"><paragraph id="f9919d53e51d4ef8bae462a052eec217">&lt;p&gt;&lt;strong&gt;IMPORTANT TECHNICAL NOTES ON CHEATING&lt;/strong&gt;&lt;/p&gt;

&lt;ul&gt;
\t&lt;li&gt;&lt;strong&gt;During the practice task, you will use the Neurophysiology (Neurofiziol\u00f3gia) software that is already running on your computer.&lt;/strong&gt;&lt;/li&gt;
\t&lt;li&gt;&lt;strong&gt;&lt;u&gt;Please do not use any other software&lt;/u&gt;&lt;/strong&gt;&lt;strong&gt;, except for the Geneious software that you will use for the other bioinformatics task.&lt;/strong&gt;&lt;/li&gt;
\t&lt;li&gt;&lt;strong&gt;Your computer records all of your activities, so if you open any other software program, or if you connect your computer to the internet, you will be excluded from this task and lose any points that you scored here.&lt;/strong&gt;&lt;/li&gt;
&lt;/ul&gt;</paragraph></box><box id="fcff48c684614d6a9b2913b9586a6802"><paragraph id="51cbc2bd22b443ac928a768b61f926c2">Please, do not start the software yet. Read the detailed descriptions on the next pages first.</paragraph></box><paragraph id="637502c95ed3481fa12df4b0773966bb">&lt;p&gt;Your task is to clarify the role of neurons involved in navigation. Navigating space is a complex problem involving many neurons. The 2014 Nobel prize was awarded to O\u2019Keefe, and the 2011 Brain prize was awarded to the Hungarian Gyorgy Buzsaki, for developing techniques to study this problem. We can observe the individual activity of hundreds of neurons. Multiple electrodes are inserted into rodent brains. After the surgery, the animal recovers and adapts to the device on its head. Next, the animal is placed into experimental situations, and measurements are performed (Figure 1).&lt;/p&gt;</paragraph><figure figid="22" id="8cf330aa099045abaefbf99b0791daf8" width="0.5"><caption id="f102c33b3a884a0cae0cd8112740e212">&lt;p&gt;Figure 1. Experimental set-up. A: Exploration field. B: Animal with electrodes after surgery. C: Camera recording the path of the animal. D: Recording unit (records potential changes of brain regions and animal movements simultaneously)&lt;/p&gt;</caption></figure><paragraph id="3f8d5dba9b1246f993b7c43692cd4d37">&lt;p&gt;Using similar techniques to you, O\u2019Keefe investigated the hippocampus and found that many of its neurons fire when the animal is standing on or passing through a given point of its exploration field. These cells were named &lt;em&gt;place cells&lt;/em&gt;. The results of such an experiment are shown on Figure 2.&lt;/p&gt;</paragraph><figure figid="22" id="9f7e8d870a9d4145a63263421f5c9cf2" width="0.3"><caption id="cc72e733109845ee9e69d2ec31e5b7ea">&lt;p&gt;Figure 2. Place cell behaviour.&lt;strong&gt; &lt;/strong&gt;Black line:&lt;strong&gt; &lt;/strong&gt;The path a rodent took as it explored a square field. Red dots: places where a certain neurone fired.&lt;/p&gt;</caption></figure><box id="a0a23b05c8f444f8a5fd54ae9b04b6c9"><paragraph id="063718c590e5420fa7a55e1218571ced">The place cells described by O\u2019Keefe are not measured in your data!</paragraph></box><part id="9ed5b2d26a2a4b0aa9824e8779d9954b">Summary of Tasks</part><paragraph id="b468ab580bd74a3b80d63313194d1796">&lt;p&gt;Your task is to analyse the function of neurons in the entorhinal cortex. The task will consist of two parts: &lt;/p&gt;

&lt;ul&gt;
\t&lt;li&gt;&lt;strong&gt;Part 1. &lt;/strong&gt;You will have to analyse recordings of 30 cells. You will identify which neurons are responsible for perceiving the same aspects of movement and/or position in space. At first, you do not have to state which aspects of movement or position each neuron responds to, you only have to identify groups of cells performing similar functions.&lt;/li&gt;
\t&lt;li&gt;&lt;strong&gt;You may only begin the second part after completing the first part.&lt;/strong&gt;&lt;/li&gt;
\t&lt;li&gt;&lt;strong&gt;Part 2. &lt;/strong&gt;When beginning the second part, the lab assistant will provide you additional sheets with a short description of the exact function of the cells in part 1. After studying these descriptions, you will have to state the precise aspects of movement/space which six neurons respond to.&lt;/li&gt;
&lt;/ul&gt;</paragraph><paragraph id="ff46cbce29e647259c2de67b9314bbb8">&lt;p&gt;The background of your experiment:&lt;/p&gt;

&lt;ol&gt;
\t&lt;li&gt;A rat implanted with electrodes was allowed to explore a 2m x 2m square field.&lt;/li&gt;
\t&lt;li&gt;We recorded the animal\u2019s movement and the activity of different neurons. &lt;/li&gt;
\t&lt;li&gt;Results are displayed in the Neurophysiology software.&lt;/li&gt;
\t&lt;li&gt;You can switch between the NEUROPHYSIOLOGY and the Geneious Prime softwares by clicking on the system tray or using the Alt+Tab key combination. &lt;/li&gt;
\t&lt;li&gt;&lt;strong&gt;Do not use the software before reading the tasks.&lt;/strong&gt;&lt;/li&gt;
&lt;/ol&gt;</paragraph><figure figid="22" id="73d4179683ca4342b833e3011e3cd1ff" width="1"><caption id="4f2bcc950eb74d488ba4a9cdca60e193">&lt;p&gt;Figure 3. Software overview.&lt;strong&gt; &lt;/strong&gt;A: Exploration field. B: Selection of neuron and run. C: Playback control. D: Statistical information on the actual run. E: Answer block&lt;/p&gt;</caption></figure><part id="190bcb8f7fd9472fac2ab2ba7fe13fda">Part 1</part><paragraph id="11653e39da694e8daf86133343a5e249">Your software displays the route of the experimental animal continuously in time. You are able to select individual neurons, and red dots appear on the animal\u2019s route when the firing of this neuron changes significantly. Each animal was recorded doing three separate runs, so three data sets are available for each neuron. (Figure 4-6).</paragraph><paragraph id="232bd240fdec4937b096ce6bd17ad0c1">In Part 1, the animal\u2019s route will fade with time, so you can only observe a portion of the rat\u2019s movements.</paragraph><figure figid="22" id="97c1f917be14495786bdfbec10452900" width="0.7"><caption id="b3ebc480dce84ac99c4ad2b30dd50157">&lt;p&gt;Figure 4. Selecting neurones and runs. In this menu, you can switch between individual neurons (\u201cneurons\u201d 1-30). Click on \u201crun\u201d and choose 1-3 from the drop-down menu.&lt;/p&gt;</caption></figure><figure figid="22" id="455e2bfe1d814d95a9889c42e6d30ae8" width="0.4"><caption id="cafddd14d36a41a9a3f6dea7caa51d8f">Figure 5. Exploration field. Blue shows route of animal, red dots show neurone activity and the arrow shows the direction of the head.</caption></figure><figure figid="22" id="895a3248ff2a44f89428447aa0f19d91" width="0.75"><caption id="59eed2113be0472fb00304794e05611e">Figure 6. Playback control. You can start (\u2018Play\u2019) or pause (\u2018Pause\u2019) the run, or you can speed up (\u2018Speed Up\u2019) or slow down (\u2018Slow Down\u2019) the playback. Using the slider, you can jump forward or backward in a run. &lt;strong&gt;Please be patient when using the slider, the software may take a few seconds to load. &lt;/strong&gt;</caption></figure><paragraph id="a2cb5ba927d04d8f97af43db52f7bbc2">&lt;p&gt;You will see aggregated data of the activity of the neuron (y-axis) plotted against aspects of its movement (Figure 7).&lt;/p&gt;

&lt;p&gt;\u2018&lt;strong&gt;Online Analysis&lt;/strong&gt;\u2019: the graphs show aggregated data collected until the current time point of the current playback. The rat\u2019s current position is represented by black lines on the graphs.&lt;/p&gt;

&lt;p&gt;\u2018&lt;strong&gt;Final Analysis&lt;/strong&gt;\u2019: the graphs show aggregated values of the entire current run. The black line returns to the 0 positions.&lt;/p&gt;

&lt;p&gt;&lt;strong&gt;FR(Hz) vs. X (cm)&lt;/strong&gt;: the firing frequency of the neuron is plotted against the animal\u2019s horizontal distance measured from the lower left corner of the exploration field.&lt;/p&gt;

&lt;p&gt;&lt;strong&gt;FR(Hz) vs. Y (cm)&lt;/strong&gt;: the firing frequency of the neuron is plotted against the animal\u2019s vertical distance measured from the lower left corner of the exploration field.&lt;/p&gt;

&lt;p&gt;&lt;strong&gt;FR (Hz) vs. Speed (cm/s)&lt;/strong&gt;: the firing frequency of the neuron is plotted against the animal\u2019s linear speed.&lt;/p&gt;

&lt;p&gt;&lt;strong&gt;LOOKING AT&lt;/strong&gt;: the firing frequency of the neuron mapped against the angle between the animal\u2019s direction of gaze and the straight line connecting the centre of the exploration field with its lower left corner.&lt;/p&gt;</paragraph><figure figid="22" id="801633d63b014553ab38d5ac8c64c5a7" width="0.4"><caption id="8afa8c87ded84b25a2328dff862152f5">Figure 7. Statistical information on the run</caption></figure><paragraph id="fc7a0a172a984a4692f4f6f67721b2ef">&lt;p&gt;You will have to use the exploration field and statistical information to determine whether each neuron has a function in the perception of movement and position characteristics. You will then have to find cells with similar function.&lt;/p&gt;

&lt;ul&gt;
\t&lt;li&gt;When you have an idea what the function of a certain neuron is, use the software to look for cells with similar function.&lt;/li&gt;
\t&lt;li&gt;&lt;strong&gt;The cells measured in your data are divided into FOUR groups based on the different characteristics of movement and position they respond to. &lt;/strong&gt;&lt;/li&gt;
\t&lt;li&gt;&lt;strong&gt;Each cell only responds to one of these four aspects of movement and position.&lt;/strong&gt;&lt;/li&gt;
\t&lt;li&gt;&lt;strong&gt;Approximately half of the 30 cells measured in your data do not respond to movement or position. These are NOT cells which you are interested in.&lt;/strong&gt;&lt;/li&gt;
\t&lt;li&gt;Your data includes at least FOUR neurons from each group.&lt;/li&gt;
\t&lt;li&gt;&lt;strong&gt;You must find THREE neurons from each group &lt;/strong&gt;
\t&lt;ul&gt;
\t\t&lt;li&gt;I.e. you must identify four groups of three neurons&lt;/li&gt;
\t\t&lt;li&gt;All the neurons in each group you enter should have the same function&lt;/li&gt;
\t\t&lt;li&gt;All the groups you enter should have different functions&lt;/li&gt;
\t&lt;/ul&gt;
\t&lt;/li&gt;
&lt;/ul&gt;</paragraph><box id="edba0e42e8fa48f1809dd7949df26894"><paragraph id="9d5a72f36b104f67b1249dfbaa28a810">&lt;strong&gt;You have a maximum of TEN ATTEMPTS at entering groups of three neurons to find all four categories&lt;/strong&gt;</paragraph></box><paragraph id="de0252a631d14d7bb4167aa8f7d0f52a">&lt;ul&gt;
\t&lt;li&gt;&lt;strong&gt;You will receive fewer marks for each attempt&lt;/strong&gt;
\t&lt;ul&gt;
\t\t&lt;li&gt;I.e. You should aim to find all four groups in your first four attempts&lt;/li&gt;
\t\t&lt;li&gt;You can use table 1 to keep track of the attempts you make. It will NOT be marked.&lt;/li&gt;
\t&lt;/ul&gt;
\t&lt;/li&gt;
&lt;/ul&gt;</paragraph><figure figid="22" id="8c0089497750474f8becb4be4d253c9f" width="0.35"><caption id="386d730d00c8450e93e15ab75ada7bcd">Table 1</caption></figure><paragraph id="c5e248acb9cf4fad9a8971d0f43d450b">&lt;p&gt;&lt;strong&gt;When you identify three neurons from the same group, enter them into the answer block (Figure 8). Repeat for each group&lt;/strong&gt;.&lt;/p&gt;</paragraph><figure figid="22" id="2f751ab3922b4abc8824cc720c0792cb" width="0.25"><caption id="9f454aaf682044ad83b428038f9a873e">Figure 8. Answer block</caption></figure><paragraph id="e542ba028b1249b98826016123673587">&lt;p&gt;Write the number of three neurons into the rectangles, then click \u201cSubmit answer\u201d. The system counts down your attempts from 10 (\u201cYou have 10 tries left\u201d)&lt;/p&gt;

&lt;p&gt;If you answer is correct you will see this box (Figure 9). Click \u201cOK\u201d and continue to work.&lt;/p&gt;</paragraph><figure figid="22" id="5270b3426eee44478d5c0da5bff49932" width="0.35"><caption id="333734a3af1e4cdf9a0dd326f56b62d6">Figure 9. Correct answer</caption></figure><paragraph id="36a079302cc14c1eae56a951405b4423">&lt;p&gt;If your answer is correct, but the neurons are in a group that you have already correctly submitted, then you will see this box (Figure 10). This will count as a false attempt and the number of remaining attempts will decrease. Click \u201cOK\u201d and continue to work.&lt;/p&gt;</paragraph><figure figid="22" id="3798db26574c4dd29e665110e58273ca" width="0.4"><caption id="81dad5e47f0247ceb4c65c30fdde0ddb">Figure 10. Re-discovery of a group</caption></figure><paragraph id="036cd42acb9241e194425cc37b6edd00">&lt;p&gt;If you submit three neurons which are not in the same group, you will see not see a pop-up box, but the number of remaining attempts will decrease, and this will count as a false attempt. Continue to work.&lt;/p&gt;</paragraph><paragraph id="411c44cd6e7f48c790180e41030c3d89">&lt;p&gt;When you correctly identify three neurons in each of the four groups, part 1 will end. Or, when you run-out of attempts, part 1 will end. You will see this box (Figure 11).&lt;/p&gt;</paragraph><figure figid="22" id="e680dfcf27894417b4b43dddcbe5618a" width="0.35"><caption id="37fc59e9678d4ac5b7810d1d297a3f3c">Figure 11. Completion of Part 1.</caption></figure><subquestion id="9fcbbed517024cc0b63811f41c44bd04" part_nr="Q" points="0.0" question_nr="4.2.1"><paragraph id="35d4e4b72d57465d8afe95b93b899ded">You MUST write the \u201cCODE: \\#\\#\\#\\#\\#\\#\\#\u201d on your ANSWER SHEET.</paragraph></subquestion><paragraph id="b7bd1f11ad1649cfb38e4b560b91ff04">&lt;p&gt;Alternatively, you &lt;strong&gt;may choose to end Part 1 &lt;/strong&gt;before you run out of attempts to save time. Click \u201cSkip this part! Show the CODE!\u201d This box will appear (Figure 12). Click \u201cYES\u201d to end the task and reveal your code. This is irreversible, you will receive marks for the correct groups you have discovered but it will count as using all ten attempts. You must write your code on your answer sheet. Click \u201cNO\u201d to return to part 1 and continue working.&lt;/p&gt;</paragraph><figure figid="22" id="30dc625d6c2d42a0b85dc94e03469973" width="0.75"><caption id="305eeee7a8904c71ae8b4d1d98e926b6">Figure 12. Giving-up</caption></figure><box id="97497f253cdf4fd695ffa98e80af264d"><paragraph id="18f5e816cb8b41f58b52f137b95e019a">&lt;p&gt;&lt;strong&gt;After completing part 1 you MUST hold up your YELLOW CARD. &lt;/strong&gt;&lt;/p&gt;

&lt;p&gt;&lt;strong&gt;An assistant will check that you have correctly written the code.&lt;/strong&gt;&lt;/p&gt;

&lt;p&gt;&lt;strong&gt;An assistant will give you INFORMATION necessary for PART 2.&lt;/strong&gt;&lt;/p&gt;</paragraph></box><pagebreak id="cb13790c102d493ab43e2bef5e6d0337" /><part id="e9cfc0da768d401089cc210737d8c62a">Part 2</part><paragraph id="0426a82852dc47409db5d13caaf0f871">Read the information given to you by the assistant describing five different groups of neurons (A, B, C, D \\&amp; E). Your software will now display the whole route of the animal without fading (Figure 13).</paragraph><figure figid="22" id="4af3742bc67a4b6aa486acc33d1467b0" width="0.4"><caption id="eb262b3b0c024aa795f0bdfe81ac8d1c">Figure 13. Exploration field for Part 2.</caption></figure><paragraph id="b3c25e8663cb4a33928ed70284cec4a2">&lt;p&gt;Six specific neurons are listed on your answer sheet (Number 23, 17, 9, 25, 3 \\&amp;amp; 11). View the Exploration fields of runs 1-3 for each of these neurons. Identify which of the five different groups each neuron belongs to.&lt;/p&gt;</paragraph><subquestion id="4f03acf199df43fdacc5bff980f83db6" part_nr="Q" points="0.0" question_nr="4.2.2"><paragraph id="5f476d23fbe34f3a9bd01230cc34c777">Put an X on your ANSWER SHEET identifying which GROUP EACH NEURON belongs to.</paragraph></subquestion><pagebreak id="442da0b99ff74504b4e6391dcbd99053" /><part id="4b03861201f943cd92e8303f33b9340c">Extra information on neuron types</part><section id="b052c4977c734a5fa5f0895500f3980a">A) Speed cells</section><paragraph id="4e1b64c929384ee1ab31e94df569c23a">&lt;p&gt;&lt;strong&gt;Speed cells\u2019 firing rate is proportional to the linear speed of the animal&lt;/strong&gt;&lt;/p&gt;

&lt;p&gt;Researchers recorded neuronal activity whilst controlling rats\u2019 running speed. Rats were put in a bottomless frame which was moved at pre-set speeds, compelling them to run (Fig. 1a). Rats either ran with gradually increasing speed (Figure 1b left, middle), or with a sharp transition in the middle (Figure 1b right).&lt;/p&gt;</paragraph><figure figid="22" id="faa319faf3394ea2960475f87c29f7e4" width="0.85"><caption id="587be75ac75a455ab57be89cf84cf604">&lt;p&gt;Figure 1. Speed set-up. Axis 1: Position in track (m). Axis 2: Firing Rate (Hz). Axis 3: Speed (cm/s)&lt;/p&gt;</caption></figure><box id="46ee3c4677534ec6a4392291400a33cb"><paragraph id="8c5fc0a471cc48a8bfa850f402f230fa">&lt;p&gt;The other cell-types were discovered with the experimental design shown in Part 1.&lt;/p&gt;

&lt;p&gt;Results shown in Figure 2.&lt;/p&gt;</paragraph></box><figure figid="22" id="a1b3c42194b54f02a8c08aa1d20a94e4" width="0.85"><caption id="d4fbde1a87834e59be1b4c7000f72b10">&lt;strong&gt;Figure 2&lt;/strong&gt;</caption></figure><section id="379d251526734f099c9dfd8c50720b1f">B) Head direction cells</section><paragraph id="f657c49601924a0281445d2ddff547ce">The diagram depicts the firing rate (distance from origin) as a function of the animal's head direction. Head direction cells fire anywhere in an environment when the animal is looking in a certain direction.</paragraph><section id="cf262745e7674f59a1b509993b57111d">C) Grid cells</section><paragraph id="3f3aed7606034e0faf8cd6d4fa81a541">Grid cells have firing areas evenly spaced in a close-packed hexagonal array across the surface of the field.</paragraph><section id="471b4330a34e43239f63f77aaec29727">D) Border cells</section><paragraph id="384abc3b6ac04e7ba67c9f6ff7a98c83">Border cells only fire when the rat is in close proximity to the wall of the field.</paragraph><section id="99ab60ad473544489d87eee82b120a5e">E) Bulk cells</section><paragraph id="99f33629f0cc44819e5d0656ce568220">These cells do not show a clear firing pattern when the animal is moving even though they are often firing during locomotion. This means we do not understand their function yet.</paragraph></question>
"""

IBO2019_DATA[
    "Experiment_2"
] = r"""

"""
